<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0053)https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/ -->
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="bpap"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC5042158/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/jig.min.css"><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Biophysics and Physicobiology"><meta name="citation_title" content="Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets"><meta name="citation_authors" content="Tsukasa Nakamura, Kentaro Tomii"><meta name="citation_date" content="2016"><meta name="citation_volume" content="13"><meta name="citation_firstpage" content="139"><meta name="citation_doi" content="10.2142/biophysico.13.0_139"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC5042158/?report=abstract"><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC5042158/"><meta name="citation_pmid" content="27924268"><meta name="DC.Title" content="Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="The Biophysical Society of Japan"><meta name="DC.Contributor" content="Tsukasa Nakamura"><meta name="DC.Contributor" content="Kentaro Tomii"><meta name="DC.Date" content="2016"><meta name="DC.Identifier" content="10.2142/biophysico.13.0_139"><meta name="DC.Language" content="en"><meta property="og:title" content="Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets"><meta property="og:type" content="article"><meta property="og:description" content="Comprehensive analysis and comparison of protein ligand-binding pockets are important to predict the ligands which bind to parts of putative ligand binding pockets. Because of the recent increase of protein structure information, such analysis demands ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png?_=0"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/ncbi.min.css" type="text/css" data-apply="standalone"><link rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/pmc.min.css" type="text/css"><link rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/pmcrefs1.min.css" type="text/css"><link rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/pmc_extras_prnt.min.css" type="text/css" media="print"><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-bpap.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><script>
				var isjq= typeof jQuery === "function"
			</script><script type="text/javascript">if (!isjq){document.write('\x3Cscript type="text/javascript" src="/core/jig/1.15.2/js/jig.min.js"\x3E\x3C/script\x3E')}{document.write('\x3Cscript type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js"\x3E\x3C/script\x3E')}</script><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/jig.nojquery.min.js"></script><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/common.min.js">//</script><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/NcbiTagServer.min.js">//</script><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/jactions.min.js">//</script><meta name="citationexporter" content="backend:&#39;https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/&#39;"><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/citationexporter.css" type="text/css"><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/MathJax.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/pdf" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/pdf/13_139.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="https://www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="8A1A6F1013903AB10000000000600060.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/4143404.css"><link type="text/css" rel="stylesheet" href="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/4157116.css" media="print"><style>#cVim-command-bar, #cVim-command-bar-mode, #cVim-command-bar-input, #cVim-command-bar-search-results,
.cVim-completion-item, .cVim-completion-item .cVim-full, .cVim-completion-item .cVim-left,
.cVim-completion-item .cVim-right {
  font-family: Helvetica, Helvetica Neue, Neue, sans-serif, monospace, Arial;
  font-size: 10pt !important;
  -webkit-font-smoothing: antialiased !important;
}

#cVim-command-bar {
  position: fixed;
  z-index: 2147483646;
  background-color: #1b1d1e;
  color: #bbb;
  display: none;
  box-sizing: content-box;
  box-shadow: 0 3px 3px rgba(0,0,0,0.4);
  left: 0;
  width: 100%;
  height: 20px;
}

#cVim-command-bar-mode {
  display: inline-block;
  vertical-align: middle;
  box-sizing: border-box;
  padding-left: 2px;
  height: 100%;
  width: 10px;
  padding-top: 2px;
  color: #888;
}

#cVim-command-bar-input {
  background-color: #1b1d1e;
  color: #bbb;
  height: 100%;
  right: 0;
  top: 0;
  width: calc(100% - 10px);
  position: absolute;
}

#cVim-command-bar-search-results {
  position: fixed;
  width: 100%;
  overflow: hidden;
  z-index: 2147483647;
  left: 0;
  box-shadow: 0 3px 3px rgba(0,0,0,0.4);
  background-color: #1c1c1c;
}

.cVim-completion-item, .cVim-completion-item .cVim-full, .cVim-completion-item .cVim-left, .cVim-completion-item .cVim-right {
  text-overflow: ellipsis;
  padding: 1px;
  display: inline-block;
  box-sizing: border-box;
  vertical-align: middle;
  overflow: hidden;
  white-space: nowrap;
}

.cVim-completion-item:nth-child(even) {
  background-color: #1f1f1f;
}

.cVim-completion-item {
  width: 100%; left: 0;
  color: #bcbcbc;
}

.cVim-completion-item[active] {
  width: 100%; left: 0;
  color: #1b1d1e;
  background-color: #f1f1f1;
}

.cVim-completion-item[active] span {
  color: #1b1d1e;
}

.cVim-completion-item .cVim-left {
  color: #fff;
  width: 37%;
}

.cVim-completion-item .cVim-right {
  font-style: italic;
  color: #888;
  width: 57%;
}


#cVim-link-container, .cVim-link-hint,
#cVim-hud, #cVim-status-bar {
  font-family: Helvetica, Helvetica Neue, Neue, sans-serif, monospace, Arial;
  font-size: 10pt !important;
  -webkit-font-smoothing: antialiased !important;
}

#cVim-link-container {
  position: absolute;
  pointer-events: none;
  width: 100%; left: 0;
  height: 100%; top: 0;
  z-index: 2147483647;
}

.cVim-link-hint {
  position: absolute;
  color: #302505 !important;
  background-color: #ffd76e !important;
  border-radius: 2px !important;
  padding: 2px !important;
  font-size: 8pt !important;
  font-weight: 500 !important;
  text-transform: uppercase !important;
  border: 1px solid #ad810c;
  display: inline-block !important;
  vertical-align: middle !important;
  text-align: center !important;
  box-shadow: 2px 2px 1px rgba(0,0,0,0.25) !important;
}

.cVim-link-hint_match {
  color: #777;
  text-transform: uppercase !important;
}


#cVim-hud {
  background-color: rgba(28,28,28,0.9);
  position: fixed !important;
  transition: right 0.2s ease-out;
  z-index: 24724289;
}

#cVim-hud span {
  padding: 2px;
  padding-left: 4px;
  padding-right: 4px;
  color: #8f8f8f;
  font-size: 10pt;
}

#cVim-frames-outline {
  position: fixed;
  width: 100%;
  height: 100%;
  left: 0;
  top: 0;
  right: 0;
  z-index: 9999999999;
  box-sizing: border-box;
  border: 3px solid yellow;
}
</style><script async="1" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style>.ncbi-alerts .ncbi-alert__shutdown-outer { position: relative; background-color: #f4e3db; border-left: 8px solid #d54309;  background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg xmlns='http://www.w3.org/2000/svg' width='126' height='126' viewBox='0 0 126 126'%3E%3Cpath fill='%231B1B1B' d='M117.18,31.592 C111.585,22.006 103.995,14.416 94.409,8.821 C84.821,3.226 74.354,0.429 63.001,0.429 C51.649,0.429 41.18,3.226 31.593,8.821 C22.006,14.415 14.416,22.005 8.821,31.592 C3.225,41.179 0.428,51.649 0.428,63 C0.428,74.351 3.226,84.82 8.82,94.408 C14.415,103.992 22.005,111.584 31.592,117.179 C41.179,122.774 51.648,125.571 63,125.571 C74.352,125.571 84.822,122.774 94.408,117.179 C103.994,111.585 111.584,103.994 117.179,94.408 C122.773,84.82 125.57,74.35 125.57,63 C125.57,51.649 122.773,41.178 117.18,31.592 Z M73.43,102.025 C73.43,102.786 73.184,103.423 72.696,103.939 C72.208,104.455 71.61,104.712 70.903,104.712 L55.26,104.712 C54.554,104.712 53.929,104.441 53.386,103.898 C52.843,103.355 52.572,102.73 52.572,102.025 L52.572,86.546 C52.572,85.84 52.843,85.215 53.386,84.672 C53.929,84.129 54.554,83.858 55.26,83.858 L70.903,83.858 C71.61,83.858 72.209,84.116 72.696,84.631 C73.184,85.149 73.43,85.785 73.43,86.546 L73.43,102.025 Z M73.266,73.999 C73.211,74.542 72.927,75.018 72.412,75.425 C71.895,75.832 71.258,76.035 70.498,76.035 L55.425,76.035 C54.664,76.035 54.012,75.832 53.469,75.425 C52.926,75.018 52.654,74.542 52.654,73.999 L51.269,23.404 C51.269,22.751 51.54,22.263 52.083,21.937 C52.627,21.503 53.279,21.285 54.039,21.285 L71.965,21.285 C72.726,21.285 73.377,21.502 73.92,21.937 C74.463,22.263 74.733,22.752 74.733,23.404 L73.266,73.999 Z'/%3E%3C/svg%3E%0A");}.ncbi-alerts .ncbi-alert__info-outer { position: relative; background-color: #e7f6f8; border-left: 8px solid #00bde3; background-image: url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='UTF-8'%3F%3E%3Csvg xmlns='http://www.w3.org/2000/svg' width='126' height='126' viewBox='0 0 126 126'%3E%3Cpath fill='%231B1B1B' d='M117.18,31.592 C111.585,22.006 103.995,14.416 94.409,8.821 C84.821,3.226 74.354,0.429 63.001,0.429 C51.649,0.429 41.179,3.226 31.593,8.821 C22.006,14.415 14.416,22.005 8.821,31.592 C3.225,41.179 0.428,51.649 0.428,63 C0.428,74.352 3.226,84.82 8.82,94.408 C14.415,103.993 22.005,111.584 31.592,117.179 C41.179,122.774 51.648,125.571 63,125.571 C74.352,125.571 84.822,122.774 94.408,117.179 C103.994,111.585 111.584,103.994 117.179,94.408 C122.773,84.821 125.57,74.351 125.57,63 C125.57,51.648 122.773,41.178 117.18,31.592 Z M52.572,16.071 C52.572,15.31 52.816,14.686 53.305,14.197 C53.794,13.709 54.419,13.464 55.179,13.464 L70.823,13.464 C71.583,13.464 72.208,13.709 72.695,14.197 C73.183,14.686 73.429,15.31 73.429,16.071 L73.429,29.107 C73.429,29.867 73.183,30.492 72.695,30.98 C72.208,31.469 71.583,31.713 70.823,31.713 L55.179,31.713 C54.419,31.713 53.794,31.469 53.305,30.98 C52.816,30.492 52.572,29.867 52.572,29.107 L52.572,16.071 Z M83.857,102.107 C83.857,102.867 83.611,103.492 83.124,103.979 C82.637,104.468 82.012,104.712 81.25,104.712 L44.75,104.712 C43.989,104.712 43.365,104.468 42.876,103.979 C42.387,103.491 42.143,102.866 42.143,102.106 L42.143,89.07 C42.143,88.308 42.387,87.685 42.876,87.196 C43.365,86.708 43.99,86.463 44.75,86.463 L52.572,86.463 L52.572,60.392 L44.75,60.392 C43.989,60.392 43.365,60.148 42.876,59.659 C42.387,59.171 42.143,58.546 42.143,57.785 L42.143,44.75 C42.143,43.989 42.387,43.365 42.876,42.876 C43.365,42.387 43.99,42.143 44.75,42.143 L70.823,42.143 C71.583,42.143 72.208,42.387 72.695,42.876 C73.183,43.365 73.429,43.989 73.429,44.75 L73.429,86.464 L81.249,86.464 C82.01,86.464 82.635,86.709 83.123,87.197 C83.61,87.685 83.856,88.31 83.856,89.071 L83.856,102.107 L83.857,102.107 Z'/%3E%3C/svg%3E");}.ncbi-alerts div[class$="-outer"]{background-position: 28px 20px;  background-size: 32px 32px;  background-repeat: no-repeat;padding:20px 20px 20px 28px;}.ncbi-alerts div[class$="-inner"]{padding-left: 52px; padding-right: 52px;}@media (max-width: 639px){.ncbi-alerts div[class$="outer"]{background-position: 15px 20px;}.ncbi-alerts div[class$="-inner"]{padding-left: 0px; padding-right: 0px; padding-top: 40px;}}.ncbi-alerts button.close{cursor: pointer;position:absolute; top: 10px; right: 20px; width: 36px; height: 32px; border:0; background-color: transparent; background-image: url("data:image/svg+xml,%3Csvg version='1.2' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' overflow='visible' preserveAspectRatio='none' viewBox='0 0 24 24' width='32' height='32'%3E%3Cg%3E%3Cpath xmlns:default='http://www.w3.org/2000/svg' id='window-close' d='M14.9,16.42c-0.13,0.13-0.33,0.14-0.47,0.01c0,0-0.01-0.01-0.01-0.01L12,14l-2.43,2.42 c-0.13,0.13-0.33,0.14-0.47,0.01c0,0-0.01-0.01-0.01-0.01L7.58,14.9c-0.13-0.13-0.14-0.33-0.01-0.47c0,0,0.01-0.01,0.01-0.01L10,12 L7.58,9.57C7.45,9.45,7.44,9.24,7.57,9.1c0,0,0.01-0.01,0.01-0.01L9.1,7.58c0.13-0.13,0.33-0.14,0.47-0.01c0,0,0.01,0.01,0.01,0.01 L12,10l2.43-2.43c0.13-0.13,0.33-0.14,0.47-0.01c0,0,0.01,0.01,0.01,0.01l1.51,1.53c0.13,0.13,0.14,0.33,0.01,0.47 c0,0-0.01,0.01-0.01,0.01L14,12l2.43,2.43c0.13,0.13,0.14,0.33,0.01,0.47c0,0-0.01,0.01-0.01,0.01L14.9,16.42L14.9,16.42z M20.84,4.49C20.53,4.17,20.1,3.99,19.66,4H4.34C3.42,4,2.67,4.75,2.67,5.67l0,0v12.66c0,0.92,0.75,1.67,1.67,1.67l0,0h15.32 c0.92,0,1.67-0.75,1.67-1.67l0,0V5.67C21.34,5.23,21.16,4.8,20.84,4.49z' style='fill: rgb(33, 33, 33);' vector-effect='non-scaling-stroke'/%3E%3C/g%3E%3C/svg%3E");background-size: 32px 32px; background-repeat:no-repeat;background-position:center center}.ncbi-alerts button.close:focus{ outline: 1px dotted #000000;}@media (max-width: 639px){.ncbi-alerts button.close{right: 0px;}}.ncbi-alerts p{margin-bottom: 0px; margin-top: 0px; font-size: 18px; line-height: 28px;}.ncbi-alerts .list-items > p{font-size: 0.94em; display: inline;}.ncbi-alerts .list-items > p a {font-weight: 700;}.ncbi-alerts .list-items > p:not(:last-child)::after{margin:0 8px;content:'|'}@media (min-width: 768px) and (max-width: 991px){.ncbi-alerts .list-items > p:nth-child(2)::after{content:'\a'!important;white-space: pre;}}@media(max-width: 767px){.ncbi-alerts .list-items > p{display:block;}.ncbi-alerts .list-items > p::after{content:''!important;}}</style><script charset="utf-8" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/11.50cce3eb2b28e5784da7.chunk.js"></script><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
</style><script charset="utf-8" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/2.816d080d0a2652e0eefc.chunk.js"></script><script charset="utf-8" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/1.212b4d44c9a5ef68f71a.chunk.js"></script></head>
    <body class="article" id="ui-ncbiexternallink-3"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="https://www.ncbi.nlm.nih.gov/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="https://www.ncbi.nlm.nih.gov/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="https://www.ncbi.nlm.nih.gov/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/emyncbi.html" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header" style="margin-bottom: 0px;">
    <div class="res_logo">
  <h1 class="img_logo"><a href="https://www.ncbi.nlm.nih.gov/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="https://www.ncbi.nlm.nih.gov/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="protfam">Protein Family Models</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:&#39;pmc-search-autocomplete&#39;,disableUrl:&#39;NcbiSearchBarAutoComplCtrl&#39;" autocomplete="off" data-sbconfig="ds:&#39;no&#39;,pjs:&#39;no&#39;,afs:&#39;yes&#39;" aria-haspopup="true" aria-autocomplete="list" role="textbox"><a class="reset" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" style="visibility: hidden;"><img src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="https://www.ncbi.nlm.nih.gov/books/NBK3825/">Help</a>
                    </li></ul></div>
</div><div class="ncbi-alerts nCoV_shutdown" id="nCoV_shutdown" role="region" aria-label="Alert" style="margin-bottom: 0px;"><div class="ncbi-alert__shutdown-outer"><div class="ncbi-alert__shutdown-inner"><p><strong>COVID-19 Information</strong></p><div class="list-items"><p><a href="https://cdc.gov/coronavirus">Public health information (CDC)</a></p><p><a href="https://covid19.nih.gov/">Research information (NIH)</a></p><p><a href="https://www.ncbi.nlm.nih.gov/sars-cov-2/">SARS-CoV-2 data (NCBI)</a></p><p><a href="https://combatcovid.hhs.gov/">Prevention and treatment information (HHS)</a></p><p><a href="https://salud.nih.gov/covid-19/">Español</a></p></div></div><button type="button" class="close" aria-controls="nCoV_shutdown" aria-label="Close Alert">&nbsp;</button></div></div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="ncbi-alerts pmc-labs-ad" id="pmc-labs-ad" role="region" aria-label="Alert" style="margin-bottom: 16px;"><div class="pmc-labs-ad-outer"><div class="pmc-labs-ad-inner"><p>Try out <a href="https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5042158/">PMC Labs</a> and tell us what you think. <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2021/06/02/pmc-labs/" target="_blank">Learn More</a>.</p></div></div><button type="button" class="close pmc-labs-ad-close" aria-controls="pmc-labs-ad" aria-label="Close Alert"></button></div>
<div class="container">
    <div id="maincontent" class="content eight_col col">

        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="https://www.ncbi.nlm.nih.gov/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="https://www.ncbi.nlm.nih.gov/pmc/journals/2913/">Biophys Physicobiol</a></li><li class="issue-page"><a class="navlink" href="https://www.ncbi.nlm.nih.gov/pmc/issues/276315/">v.13; 2016</a></li><li class="accid">PMC5042158</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/logo-bpap.gif" alt="Logo of bpap" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area target="pmc_ext" rel="noopener noreferrer" shape="rect" coords="0,0,500,75" alt="Biophysics and Physicobiology" title="Biophysics and Physicobiology" href="https://www.jstage.jst.go.jp/browse/biophysico" ref="reftype=publisher&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: [&#39;h2&#39;], headingExclude: &#39;:hidden,.nomenu&#39;" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span role="menubar"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" role="menuitem" aria-expanded="false" aria-haspopup="true">Biophys Physicobiol.</a></span> 2016; 13: 139–147. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2016 Jul 14. </span>  <span class="doi"><span>doi:&nbsp;</span><a href="https://dx.doi.org/10.2142%2Fbiophysico.13.0_139" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.2142/biophysico.13.0_139</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC5042158</span></div><div class="fm-citation-pmid">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27924268">27924268</a></div></div></div><h1 class="content-title">Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Nakamura%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27924268" class="affpopup" co-rid="_co_idm140309106626880" co-class="co-affbox">Tsukasa Nakamura</a><sup>1,</sup><sup>2</sup> and  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Tomii%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27924268" class="affpopup" co-rid="_co_idm140309106624656" co-class="co-affbox">Kentaro Tomii</a><sup>1,</sup><sup>2,</sup><sup>3</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140309106626880"><h3 class="no_margin">Tsukasa Nakamura</h3><p><sup>1</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa-shi, Chiba 277-8562, Japan</p><p><sup>2</sup>Artificial Intelligence Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Koto-ku, Tokyo 135-0064, Japan</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Nakamura%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27924268">Tsukasa Nakamura</a></div></div><div id="_co_idm140309106624656"><h3 class="no_margin">Kentaro Tomii</h3><p><sup>1</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa-shi, Chiba 277-8562, Japan</p><p><sup>2</sup>Artificial Intelligence Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Koto-ku, Tokyo 135-0064, Japan</p><p><sup>3</sup>Biotechnology Research Institute for Drug Discovery, National Institute of Advanced Industrial Science and Technology (AIST), Koto-ku, Tokyo 135-0064, Japan</p><div>Find articles by <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Tomii%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=27924268">Kentaro Tomii</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers fm-copyright-license"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="pmctoggle" rid="idm140309109230400_ai">Author information</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="pmctoggle" rid="idm140309109230400_an">Article notes</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="pmctoggle" rid="idm140309109230400_cpl">Copyright and License information</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/disclaimer/" style="margin-left: 1em">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140309109230400_ai" style="display:none"><div class="fm-affl" lang="en" id="af1-13_139"><sup>1</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa-shi, Chiba 277-8562, Japan</div><div class="fm-affl" lang="en" id="af2-13_139"><sup>2</sup>Artificial Intelligence Research Center, National Institute of Advanced Industrial Science and Technology (AIST), Koto-ku, Tokyo 135-0064, Japan</div><div class="fm-affl" lang="en" id="af3-13_139"><sup>3</sup>Biotechnology Research Institute for Drug Discovery, National Institute of Advanced Industrial Science and Technology (AIST), Koto-ku, Tokyo 135-0064, Japan</div><div id="c1-13_139">Corresponding author: Kentaro Tomii, National Institute of Advanced Industrial Science and Technology (AIST), 2-4-7 Aomi, Koto-ku, Tokyo 135-0064, Japan. e-mail: <a href="mailto:dev@null" data-email="pj.og.tsia@iimot-k" class="oemail">pj.og.tsia@iimot-k</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140309109230400_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2016 Jan 12; Accepted 2016 Jun 6.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140309109230400_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="https://www.ncbi.nlm.nih.gov/pmc/about/copyright/">Copyright</a> © 2016 The Biophysical Society of Japan</div><div class="license half_rhythm">This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/bin/13_139_1.pdf" data-ga-action="click_feat_suppl">13_139_1.pdf</a><span style="color:gray"> (752K)</span></div><div class="small guid">GUID:&nbsp;CD96512E-B7AF-4EC4-B544-8E65366CBFBD</div></div></dd></dl></div><div id="idm140309110378112" lang="en" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140309110378112title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Comprehensive analysis and comparison of protein ligand-binding pockets are important to predict the ligands which bind to parts of putative ligand binding pockets. Because of the recent increase of protein structure information, such analysis demands a fast and efficient method for comparing ligand binding pockets. Previously we proposed a fast alignment-free method based on a simple representation of a ligand binding pocket with one 11-dimensional vector, which is suitable for such analysis. Based on that method, we conducted this study to expand and revise similarity measures of binding pockets and to investigate the effects of those modifications with two datasets for improving the ability to detect similar binding pockets. The new method exhibits higher detection performance of similar binding pockets than the previous methods and another existing accurate alignment-dependent method: <em>APoc</em>. Results also show that the effects of the modifications depend on the difficulty of the dataset, implying some avenues for methods of improvement.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">multidimensional scaling, alignment-free comparison, triangle descriptor, large-scale comparison</span></div></div><div id="idm140309109474160" class="tsec sec"><h2 class="headless nomenu"></h2><p id="__p2" class="p p-first">Elucidation of interactions between proteins and ligands such as small molecules is particularly important. They can be expected to facilitate the functional elucidation of proteins, with relations to metabolism, drug discovery, and drug repositioning. With the recent increase of a database of known protein structure, the Protein Data Bank (PDB) [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b1-13_139" rid="b1-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930652" role="button" aria-expanded="false" aria-haspopup="true">1</a>], we have huge amounts of structural information for approximately 300,000 known and 5.2 million unknown (estimated using a pocket identification program) ligand-binding pockets [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b2-13_139" rid="b2-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930668" role="button" aria-expanded="false" aria-haspopup="true">2</a>]. Consequently, comprehensive comparison and classification of both known and predicted protein ligand-binding pockets provide important insights into predicting ligands and drug discovery. For such a comprehensive analysis, a fast pocket comparison method is extremely useful. Indeed, various approaches have already been proposed [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b3-13_139" rid="b3-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930666" role="button" aria-expanded="false" aria-haspopup="true">3</a>].</p><p id="__p3">Pocket comparison methods are divisible into two classes, i.e., alignment-dependent and alignment-free methods [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b4-13_139" rid="b4-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930646" role="button" aria-expanded="false" aria-haspopup="true">4</a>]. Although alignment-dependent methods for pocket comparison perform structural alignment of binding residues, alignment-free methods are independent of the structural alignment. Such methods often use descriptors that represent binding residues in a pocket. Both methods have been developed to be applicable to large-scale comparison of binding pockets. For instance, Gao <em>et al.</em> developed a fast alignment-dependent method called <em>APoc</em> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b5-13_139" rid="b5-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930656" role="button" aria-expanded="false" aria-haspopup="true">5</a>]. They argued that alignment-dependent methods are “generally more accurate, albeit slower than alignment-free methods.” However, although alignment-free methods are unable to provide information of matched residues, they are efficient in terms of computational time. Consequently, these methods enable comparison of known binding pockets with numerous predicted ligand-binding pockets estimated using a pocket detection program. Furthermore, alignment-free methods are compatible with analysis of “flexible” binding pockets, and are readily applicable to binding pockets comprising multiple protein chains. It remains difficult to apply alignment-dependent methods directly to such pockets.</p><p id="__p4">Considering these reasons, we assume that alignment-free methods can particularly contribute to protein–ligand interaction prediction and can enable the prediction of protein function. Therefore, we developed an alignment-free method that enables us to perform exhaustive comparison of both known and predicted ligand-binding pockets of 1,000,000 order [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b6-13_139" rid="b6-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930665" role="button" aria-expanded="false" aria-haspopup="true">6</a>], and to develop a database called PoSSuM that includes the comparison results [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b2-13_139" rid="b2-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930669" role="button" aria-expanded="false" aria-haspopup="true">2</a>,<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b7-13_139" rid="b7-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930670" role="button" aria-expanded="false" aria-haspopup="true">7</a>]. Recently, we also proposed a fast method based on a simple representation, an 11-dimensional vector, of a ligand-binding pocket using a triangle descriptor defined by a set of three amino acids in a pocket and multidimensional scaling (MDS) [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b8-13_139" rid="b8-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930659" role="button" aria-expanded="false" aria-haspopup="true">8</a>]. In this method, the vector representation of a ligand-binding pocket is obtained using the linear combination of the occurrence frequency of triangles using their coordinates in a metric space. Using this method, one can calculate the similarity between two ligand-binding pockets merely by calculating the inner product of two reduced vectors.</p><p id="__p5" class="p">For this study presented here, we sought to revise the definition of the triangle descriptor of ligand-binding pockets for improving the discriminative ability of our method. To define new similarity measures of binding pockets, we used an amino acid similarity matrix instead of the distance matrix used in our previous method to present physicochemical similarity between amino acids in pockets. We expanded the classes to consider the geometrical similarity between edges of the triangle descriptor, and modified the definition of those classes. In addition to these points, we also examined use of the increased number, instead of 11 used in our previous method, of dimensions in the results of MDS. The new method belongs to the alignment-free classes, exhibiting higher detection performance of similar binding pockets than our previous method and an existing fast sequence order-independent structural alignment method: <em>APoc</em>. This report describes that the effects of the modifications depend on the dataset difficulty. These results suggest some avenues for future development of similarity measures between binding pockets for improvement of pocket comparison methods.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Materials and Methods</h2><div id="__sec2" class="sec sec-first"><h3 id="__sec2title">New similarity measures for ligand-binding pocket comparison</h3><p id="__p6" class="p p-first-last">We have proposed a simple representation, an 11-dimensional vector, of a ligand-binding pocket using triangle descriptor defined by a set of three amino acids in a pocket and MDS [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b8-13_139" rid="b8-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930650" role="button" aria-expanded="false" aria-haspopup="true">8</a>]. For this study, we intend to expand and revise similarity measures of our method and to elucidate their effects for improvement of the detection ability of similar binding pockets. We mainly emphasize and modify the following two points based on our previous method. i) Whereas our previous method used Miyata’s amino acid distance matrix [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b9-13_139" rid="b9-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930664" role="button" aria-expanded="false" aria-haspopup="true">9</a>] to represent physicochemical distance or dissimilarity between two amino acids to be compared, we employ an amino acid similarity matrix to present physicochemical similarity between two amino acids in our new method presented here. ii) To represent the geometrical dissimilarity between edges of the triangle descriptor, in our previous method, we considered edges that correspond to the Cα–Cα distances of residue pairs of 1.0 Å to 13.6 Å, and classified them into five classes at 2.2 Å intervals. In our new methods, we extended Cα–Cα distances to be considered, ranging from 1.0 Å to 15.8 Å, and added a class of edges. Edges were classified into 6 classes at intervals of 2.2 Å. We assigned them Roman numerals (I, II, III, IV, V, and VI) in ascending order (hereinafter designated as ‘interval set α’). We also investigated the effects of revising the interval distances which affect the classes of edges. We modified the intervals of 6 edge classes from 2.2 Å each to 1.0, 4.0, 6.36, 8.72, 11.08, 13.44, and 15.8 Å (hereinafter, ‘interval set β’). According to these expansions and revisions, the definition of similarity between two triangle types is modified slightly (see below <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#fd1-13_139" rid="fd1-13_139" class=" disp-formula">eq. (1)</a>). In addition to these points, we also examined the use of a higher number of dimensions, instead of the 11 used in our previous method, in the results of MDS. Detailed explanations of modifications in our new methods are described below.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Enumerating possible triangle types</h3><p id="__p7" class="p p-first-last">First, similarly to our previous study, we enumerated all possible triangle types. In this study, we eventually increased the classes of edge labels to 6 labels from the 5 labels used in the previous study. However, vertex labels were the same as those used previously: 20 labels with one letter amino acid of 20 types. In all, we have the 295,240 triangle types for the 6 labels (and the 171,700 triangle types for the 5 labels [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b8-13_139" rid="b8-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930648" role="button" aria-expanded="false" aria-haspopup="true">8</a>]).</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Definition of similarity between two triangle types</h3><p id="__p8" class="p p-first">In our new method, for two triangle types of <em>p</em> and <em>q</em>, we defined the similarity <em>s</em><em><sub>pq</sub></em> consisting of two terms. They respectively denote physicochemical and geometrical similarity, as</p><div class="disp-formula" id="fd1-13_139"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="m1" overflow="scroll" role="math" style="width: 26.767em; display: inline-block;"><span style="display: inline-block; position: relative; width: 21.575em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(1.313em, 1021.52em, 5.648em, -999.997em); top: -3.727em; left: 0.003em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mrow" id="MathJax-Span-4"><span class="msub" id="MathJax-Span-5"><span style="display: inline-block; position: relative; width: 1.162em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-6"><span class="mi" id="MathJax-Span-7" style="font-family: STIXGeneral-Italic;">s</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mrow" id="MathJax-Span-8"><span class="mi" id="MathJax-Span-9" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">p</span><span class="mi" id="MathJax-Span-10" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">q</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-11" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">≡</span><span class="mtext" id="MathJax-Span-12" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">max</span><span class="mrow" id="MathJax-Span-13" style="padding-left: 0.204em;"><span class="mo" id="MathJax-Span-14" style="vertical-align: 2.271em;"><span style="display: inline-block; position: relative; width: 0.456em; height: 0px;"><span style="position: absolute; font-family: STIXSizeOneSym; top: -3.274em; left: 0.003em;">⎡<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXSizeOneSym; top: -0.3em; left: 0.003em;">⎣<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="font-family: STIXSizeOneSym; position: absolute; top: -2.568em; left: 0.003em;">⎢<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="font-family: STIXSizeOneSym; position: absolute; top: -1.812em; left: 0.003em;">⎢<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="font-family: STIXSizeOneSym; position: absolute; top: -1.056em; left: 0.003em;">⎢<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-15"><span class="mtable" id="MathJax-Span-16" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 15.527em; height: 0px;"><span style="position: absolute; clip: rect(2.623em, 1015.48em, 6.807em, -999.997em); top: -4.887em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 15.527em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1014.42em, 4.337em, -999.997em); top: -5.441em; left: 50%; margin-left: -7.205em;"><span class="mtd" id="MathJax-Span-17"><span class="mrow" id="MathJax-Span-18"><span class="mrow" id="MathJax-Span-19"><span class="mi" id="MathJax-Span-20" style="font-family: STIXGeneral-Italic;">r<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mo" id="MathJax-Span-21" style="font-family: STIXGeneral-Regular;">(</span><span class="msub" id="MathJax-Span-22"><span style="display: inline-block; position: relative; width: 1.767em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-23"><span class="mi" id="MathJax-Span-24" style="font-family: STIXGeneral-Italic;">m</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mrow" id="MathJax-Span-25"><span class="mi" id="MathJax-Span-26" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">A</span><span class="mi" id="MathJax-Span-27" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-28" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">+</span><span class="msub" id="MathJax-Span-29" style="padding-left: 0.255em;"><span style="display: inline-block; position: relative; width: 1.666em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-30"><span class="mi" id="MathJax-Span-31" style="font-family: STIXGeneral-Italic;">m</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mrow" id="MathJax-Span-32"><span class="mi" id="MathJax-Span-33" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">B</span><span class="mi" id="MathJax-Span-34" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-35" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">+</span><span class="msub" id="MathJax-Span-36" style="padding-left: 0.255em;"><span style="display: inline-block; position: relative; width: 1.767em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-37"><span class="mi" id="MathJax-Span-38" style="font-family: STIXGeneral-Italic;">m</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.708em;"><span class="mrow" id="MathJax-Span-39"><span class="mi" id="MathJax-Span-40" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-41" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">F<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-42" style="font-family: STIXGeneral-Regular;">)</span><span class="mo" id="MathJax-Span-43" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">+</span><span class="mo" id="MathJax-Span-44" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">(</span><span class="mn" id="MathJax-Span-45" style="font-family: STIXGeneral-Regular;">1</span><span class="mo" id="MathJax-Span-46" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">−</span><span class="mi" id="MathJax-Span-47" style="font-family: STIXGeneral-Italic; padding-left: 0.255em;">r<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mo" id="MathJax-Span-48" style="font-family: STIXGeneral-Regular;">)</span><span class="mo" id="MathJax-Span-49" style="font-family: STIXGeneral-Regular;">(</span><span class="mo" id="MathJax-Span-50" style="font-family: STIXGeneral-Regular;">−</span><span class="mn" id="MathJax-Span-51" style="font-family: STIXGeneral-Regular;">1</span><span class="mo" id="MathJax-Span-52" style="font-family: STIXGeneral-Regular;">)</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.178em, 1015.48em, 4.337em, -999.997em); top: -3.929em; left: 50%; margin-left: -7.76em;"><span class="mtd" id="MathJax-Span-53"><span class="mrow" id="MathJax-Span-54"><span class="mrow" id="MathJax-Span-55"><span class="mo" id="MathJax-Span-56" style="font-family: STIXGeneral-Regular;">(</span><span class="mi" id="MathJax-Span-57" style="font-family: STIXGeneral-Italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.154em;"></span></span><span class="mo" id="MathJax-Span-58" style="font-family: STIXGeneral-Regular;">(</span><span class="mi" id="MathJax-Span-59" style="font-family: STIXGeneral-Italic;">A</span><span class="mi" id="MathJax-Span-60" style="font-family: STIXGeneral-Italic;">B</span><span class="mo" id="MathJax-Span-61" style="font-family: STIXGeneral-Regular;">,</span><span class="mi" id="MathJax-Span-62" style="font-family: STIXGeneral-Italic; padding-left: 0.204em;">D</span><span class="mi" id="MathJax-Span-63" style="font-family: STIXGeneral-Italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mo" id="MathJax-Span-64" style="font-family: STIXGeneral-Regular;">)</span><span class="mo" id="MathJax-Span-65" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">+</span><span class="mi" id="MathJax-Span-66" style="font-family: STIXGeneral-Italic; padding-left: 0.255em;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.154em;"></span></span><span class="mo" id="MathJax-Span-67" style="font-family: STIXGeneral-Regular;">(</span><span class="mi" id="MathJax-Span-68" style="font-family: STIXGeneral-Italic;">B</span><span class="mi" id="MathJax-Span-69" style="font-family: STIXGeneral-Italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mo" id="MathJax-Span-70" style="font-family: STIXGeneral-Regular;">,</span><span class="mi" id="MathJax-Span-71" style="font-family: STIXGeneral-Italic; padding-left: 0.204em;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-72" style="font-family: STIXGeneral-Italic;">F<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-73" style="font-family: STIXGeneral-Regular;">)</span><span class="mo" id="MathJax-Span-74" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">+</span><span class="mi" id="MathJax-Span-75" style="font-family: STIXGeneral-Italic; padding-left: 0.255em;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.154em;"></span></span><span class="mo" id="MathJax-Span-76" style="font-family: STIXGeneral-Regular;">(</span><span class="mi" id="MathJax-Span-77" style="font-family: STIXGeneral-Italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mi" id="MathJax-Span-78" style="font-family: STIXGeneral-Italic;">A</span><span class="mo" id="MathJax-Span-79" style="font-family: STIXGeneral-Regular;">,</span><span class="mi" id="MathJax-Span-80" style="font-family: STIXGeneral-Italic; padding-left: 0.204em;">F<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mi" id="MathJax-Span-81" style="font-family: STIXGeneral-Italic;">D</span><span class="mo" id="MathJax-Span-82" style="font-family: STIXGeneral-Regular;">)</span><span class="mo" id="MathJax-Span-83" style="font-family: STIXGeneral-Regular;">)</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1008.47em, 4.337em, -999.997em); top: -2.467em; left: 50%; margin-left: -4.231em;"><span class="mtd" id="MathJax-Span-84"><span class="mrow" id="MathJax-Span-85"><span class="mrow" id="MathJax-Span-86"><span class="mo" id="MathJax-Span-87" style="font-family: STIXGeneral-Regular;">:</span><span class="mn" id="MathJax-Span-88" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">6</span><span class="mi" id="MathJax-Span-89" style="font-family: STIXGeneral-Regular;"><span style="margin-left: 0.154em;"></span></span><span class="mtext" id="MathJax-Span-90" style="font-family: STIXGeneral-Regular;">way<span style="margin-left: 0.154em;"></span>superposition</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.892em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-91" style="vertical-align: 2.271em;"><span style="display: inline-block; position: relative; width: 0.456em; height: 0px;"><span style="position: absolute; font-family: STIXSizeOneSym; top: -3.274em; left: 0.003em;">⎤<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXSizeOneSym; top: -0.3em; left: 0.003em;">⎦<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="font-family: STIXSizeOneSym; position: absolute; top: -2.568em; left: 0.003em;">⎥<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="font-family: STIXSizeOneSym; position: absolute; top: -1.812em; left: 0.003em;">⎥<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="font-family: STIXSizeOneSym; position: absolute; top: -1.056em; left: 0.003em;">⎥<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-92" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">.</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.732em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -2.247em; border-left: 0px solid; width: 0px; height: 5.066em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="m1" overflow="scroll"><mrow><mrow><msub><mrow><mi>s</mi></mrow><mrow><mi>p</mi><mi>q</mi></mrow></msub><mo>≡</mo><mtext>max</mtext><mrow><mo>[</mo><mrow><mtable><mtr><mtd><mrow><mi>r</mi><mo stretchy="false">(</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>+</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>B</mi><mi>E</mi></mrow></msub><mo>+</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>C</mi><mi>F</mi></mrow></msub><mo stretchy="false">)</mo><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>-</mo><mi>r</mi><mo stretchy="false">)</mo><mo stretchy="false">(</mo><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mtd></mtr><mtr><mtd><mrow><mo stretchy="false">(</mo><mi>f</mi><mo stretchy="false">(</mo><mi>A</mi><mi>B</mi><mo>,</mo><mi>D</mi><mi>E</mi><mo stretchy="false">)</mo><mo>+</mo><mi>f</mi><mo stretchy="false">(</mo><mi>B</mi><mi>C</mi><mo>,</mo><mi>E</mi><mi>F</mi><mo stretchy="false">)</mo><mo>+</mo><mi>f</mi><mo stretchy="false">(</mo><mi>C</mi><mi>A</mi><mo>,</mo><mi>F</mi><mi>D</mi><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></mtd></mtr><mtr><mtd><mrow><mo>:</mo><mn>6</mn><mi> </mi><mtext>way superposition</mtext></mrow></mtd></mtr></mtable></mrow><mo>]</mo></mrow><mo>.</mo></mrow></mrow></math></script></div><div class="l">(1)</div></div><p id="__p9">Therein, <em>m</em><em><sub>XY</sub></em> represents a physicochemical similarity between two amino acids <em>X</em> and <em>Y</em>, defined with an amino acid substitution matrix. For this study, we used the PAM50 matrix [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b10-13_139" rid="b10-13_139" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">10</a>], which is not rounded after a decimal point, because we assumed that residues consisting of a ligand-binding pocket are conservative for substituting amino acids. Also, the not-rounded PAM50 matrix yielded slightly better performance than the rounded (data not shown). In addition, <em>A</em>, <em>B</em>, and <em>C</em> respectively denote the vertices of the triangle type <em>p</em>; <em>D</em>, <em>E</em>, and <em>F</em> respectively denote those of triangle type <em>q</em>. <em>r</em> is a weighting factor, ranging from 0 to 1, for physicochemical and geometrical similarity terms in this equation. <em>AB</em>, <em>BC</em>, and <em>CA</em> denote the edges of the triangle type <em>p</em>; <em>DE</em>, <em>EF</em>, and <em>FD</em> denote those of triangle type <em>q</em>. Function <em>f</em>, which represents the geometrical dissimilarity between two edges <em>X</em> and <em>Y</em>, is defined as</p><div class="disp-formula" id="fd2-13_139"><div class="f"><em>f</em>(<em>e</em><em>d</em><em>g</em><em>e</em><em>X</em>,&nbsp;<em>e</em><em>d</em><em>g</em><em>e</em><em>Y</em>)&nbsp;≡&nbsp;<em> </em>|value of the class for edgeX -&nbsp;value of the class for edgeY|.</div></div><p id="__p10" class="p p-last">In this definition, the value of a class is given according to the assigned numerals for a class. For example, function <em>f</em> gives 4 when edge <em>X</em> belongs to class I and edge <em>Y</em> belongs to class V. Then, <em>f</em> is summed up for three edges and multiplied by −1 for converting dissimilarity to similarity (see <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#fd1-13_139" rid="fd1-13_139" class=" disp-formula">eq. (1)</a>). We regarded the maximum value of <em>s</em><em><sub>pq</sub></em> for all possible ways of superposition of triangle types considering rotation and reflection as similarity <em>s</em><em><sub>pq</sub></em> for two triangle types <em>p</em> and <em>q</em>.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Multidimensional Scaling (MDS)</h3><p id="__p11" class="p p-first">To execute MDS, we calculated the similarities for all possible pairs of 295,240 triangle types based on the formulation described above. We were able to obtain a similarity matrix <strong><em>S</em></strong> between triangle types as a square matrix of order 295,240. We assumed a model by which the similarity between triangle types corresponds to the inner product, and used MDS to obtain the coordinates of each triangle type in a high-dimensional space (it is also called kernel PCA, if the similarity function is guaranteed to have positive (semi-) definite property) [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b11-13_139" rid="b11-13_139" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11</a>]. The procedures used for this study are summarized briefly as follows. First centering is performed over the previously described similarity matrix <strong><em>S</em></strong> to obtain the centered similarity matrix <strong><em><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="MathJax-Span-93" role="math" style="width: 0.747em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px; font-size: 126%;"><span style="position: absolute; clip: rect(-0.097em, 1000.6em, 1.143em, -999.998em); top: -0.99em; left: 0.002em;"><span class="mrow" id="MathJax-Span-94"><span class="mover" id="MathJax-Span-95"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px;"><span style="position: absolute; clip: rect(3.227em, 1000.5em, 4.169em, -999.998em); top: -4.015em; left: 0.052em;"><span class="mi" id="MathJax-Span-96" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span><span style="position: absolute; clip: rect(3.127em, 1000.55em, 3.574em, -999.998em); top: -4.214em; left: 0.052em;"><span class="mo" id="MathJax-Span-97" style=""><span style="font-family: STIXSizeOneSym;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.995em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.316em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math><mover accent="true"><mi>S</mi><mo>̃</mo></mover></math></script></em></strong> because we want to obtain, eventually, those coordinates which have zero mean. The element of the centered similarity matrix <strong><em><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="MathJax-Span-98" role="math" style="width: 0.747em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px; font-size: 126%;"><span style="position: absolute; clip: rect(-0.097em, 1000.6em, 1.143em, -999.998em); top: -0.99em; left: 0.002em;"><span class="mrow" id="MathJax-Span-99"><span class="mover" id="MathJax-Span-100"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px;"><span style="position: absolute; clip: rect(3.227em, 1000.5em, 4.169em, -999.998em); top: -4.015em; left: 0.052em;"><span class="mi" id="MathJax-Span-101" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span><span style="position: absolute; clip: rect(3.127em, 1000.55em, 3.574em, -999.998em); top: -4.214em; left: 0.052em;"><span class="mo" id="MathJax-Span-102" style=""><span style="font-family: STIXSizeOneSym;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.995em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.316em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math><mover accent="true"><mi>S</mi><mo>̃</mo></mover></math></script></em></strong> is obtainable as</p><div class="disp-formula" id="fd3-13_139"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style=""><nobr><span class="math" id="m3" overflow="scroll" role="math" style="width: 32.311em; display: inline-block;"><span style="display: inline-block; position: relative; width: 26.061em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(1.011em, 1026.01em, 3.531em, -999.997em); top: -2.518em; left: 0.003em;"><span class="mrow" id="MathJax-Span-104"><span class="mrow" id="MathJax-Span-105"><span class="mrow" id="MathJax-Span-106"><span class="msub" id="MathJax-Span-107"><span style="display: inline-block; position: relative; width: 1.061em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1000.61em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-108"><span class="mover" id="MathJax-Span-109"><span style="display: inline-block; position: relative; width: 0.607em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.51em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mi" id="MathJax-Span-110" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.077em, 1000.56em, 3.531em, -999.997em); top: -4.181em; left: 0.053em;"><span class="mo" id="MathJax-Span-111" style=""><span style="font-family: STIXSizeOneSym;">˜</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.607em;"><span class="mrow" id="MathJax-Span-112"><span class="mi" id="MathJax-Span-113" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">i</span><span class="mi" id="MathJax-Span-114" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-115" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">=</span><span class="msub" id="MathJax-Span-116" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 0.96em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.51em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-117"><span class="mi" id="MathJax-Span-118" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-119"><span class="mi" id="MathJax-Span-120" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">i</span><span class="mi" id="MathJax-Span-121" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-122" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">−</span><span class="mfrac" id="MathJax-Span-123" style="padding-left: 0.255em;"><span style="display: inline-block; position: relative; width: 0.859em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.178em, 1000.41em, 4.136em, -999.997em); top: -4.635em; left: 50%; margin-left: -0.249em;"><span class="mn" id="MathJax-Span-124" style="font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.274em; left: 50%; margin-left: -0.35em;"><span class="mi" id="MathJax-Span-125" style="font-family: STIXGeneral-Italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1000.86em, 1.212em, -999.997em); top: -1.258em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 0.859em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-126" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 2.573em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1001.21em, 4.589em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-127" style="font-family: STIXSizeOneSym; vertical-align: -0.502em;">∑</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.38em, 1000.61em, 4.136em, -999.997em); top: -4.685em; left: 1.313em;"><span class="mi" id="MathJax-Span-128" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.38em, 1001.26em, 4.136em, -999.997em); top: -3.475em; left: 1.313em;"><span class="mrow" id="MathJax-Span-129"><span class="mi" id="MathJax-Span-130" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">a</span><span class="mo" id="MathJax-Span-131" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">=</span><span class="mn" id="MathJax-Span-132" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-133" style="padding-left: 0.204em;"><span class="msub" id="MathJax-Span-134"><span style="display: inline-block; position: relative; width: 1.111em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.51em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-135"><span class="mi" id="MathJax-Span-136" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-137"><span class="mi" id="MathJax-Span-138" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">i</span><span class="mi" id="MathJax-Span-139" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">a</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-140" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">−</span><span class="mfrac" id="MathJax-Span-141" style="padding-left: 0.255em;"><span style="display: inline-block; position: relative; width: 0.859em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.178em, 1000.41em, 4.136em, -999.997em); top: -4.635em; left: 50%; margin-left: -0.249em;"><span class="mn" id="MathJax-Span-142" style="font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.274em; left: 50%; margin-left: -0.35em;"><span class="mi" id="MathJax-Span-143" style="font-family: STIXGeneral-Italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1000.86em, 1.212em, -999.997em); top: -1.258em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 0.859em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-144" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 2.573em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1001.21em, 4.589em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-145" style="font-family: STIXSizeOneSym; vertical-align: -0.502em;">∑</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.38em, 1000.61em, 4.136em, -999.997em); top: -4.685em; left: 1.313em;"><span class="mi" id="MathJax-Span-146" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.38em, 1001.26em, 4.136em, -999.997em); top: -3.475em; left: 1.313em;"><span class="mrow" id="MathJax-Span-147"><span class="mi" id="MathJax-Span-148" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">a</span><span class="mo" id="MathJax-Span-149" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">=</span><span class="mn" id="MathJax-Span-150" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-151" style="padding-left: 0.204em;"><span class="msub" id="MathJax-Span-152"><span style="display: inline-block; position: relative; width: 1.111em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.51em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-153"><span class="mi" id="MathJax-Span-154" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-155"><span class="mi" id="MathJax-Span-156" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">a</span><span class="mi" id="MathJax-Span-157" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">j<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-158" style="font-family: STIXGeneral-Regular; padding-left: 0.255em;">+</span><span class="mfrac" id="MathJax-Span-159" style="padding-left: 0.255em;"><span style="display: inline-block; position: relative; width: 1.363em; height: 0px; margin-right: 0.103em; margin-left: 0.103em;"><span style="position: absolute; clip: rect(3.178em, 1000.41em, 4.136em, -999.997em); top: -4.635em; left: 50%; margin-left: -0.249em;"><span class="mn" id="MathJax-Span-160" style="font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.976em, 1001.21em, 4.136em, -999.997em); top: -3.173em; left: 50%; margin-left: -0.602em;"><span class="mrow" id="MathJax-Span-161"><span class="msup" id="MathJax-Span-162"><span style="display: inline-block; position: relative; width: 1.212em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-163"><span class="mi" id="MathJax-Span-164" style="font-family: STIXGeneral-Italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.383em; left: 0.809em;"><span class="mn" id="MathJax-Span-165" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(0.859em, 1001.36em, 1.212em, -999.997em); top: -1.258em; left: 0.003em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0.003em; border-top: 1.3px solid; width: 1.363em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span><span class="msubsup" id="MathJax-Span-166" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 2.573em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1001.21em, 4.589em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-167" style="font-family: STIXSizeOneSym; vertical-align: -0.502em;">∑</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.38em, 1000.61em, 4.136em, -999.997em); top: -4.685em; left: 1.313em;"><span class="mi" id="MathJax-Span-168" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.38em, 1001.26em, 4.136em, -999.997em); top: -3.475em; left: 1.313em;"><span class="mrow" id="MathJax-Span-169"><span class="mi" id="MathJax-Span-170" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">a</span><span class="mo" id="MathJax-Span-171" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">=</span><span class="mn" id="MathJax-Span-172" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-173" style="padding-left: 0.204em;"><span class="msubsup" id="MathJax-Span-174"><span style="display: inline-block; position: relative; width: 2.573em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1001.21em, 4.589em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mo" id="MathJax-Span-175" style="font-family: STIXSizeOneSym; vertical-align: -0.502em;">∑</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.38em, 1000.61em, 4.136em, -999.997em); top: -4.685em; left: 1.313em;"><span class="mi" id="MathJax-Span-176" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.329em, 1001.26em, 4.136em, -999.997em); top: -3.475em; left: 1.313em;"><span class="mrow" id="MathJax-Span-177"><span class="mi" id="MathJax-Span-178" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">b</span><span class="mo" id="MathJax-Span-179" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">=</span><span class="mn" id="MathJax-Span-180" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mrow" id="MathJax-Span-181" style="padding-left: 0.204em;"><span class="msub" id="MathJax-Span-182"><span style="display: inline-block; position: relative; width: 1.263em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.51em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-183"><span class="mi" id="MathJax-Span-184" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-185"><span class="mi" id="MathJax-Span-186" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">a</span><span class="mi" id="MathJax-Span-187" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">b</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span class="mo" id="MathJax-Span-188" style="font-family: STIXGeneral-Regular;">.</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.523em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.122em; border-left: 0px solid; width: 0px; height: 2.816em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="m3" overflow="scroll"><mrow><mrow><msub><mrow><mover accent="true"><mi>S</mi><mo>˜</mo></mover></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><msub><mrow><mi>S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>-</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><msubsup><mo>∑</mo><mrow><mi>a</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><mrow><msub><mrow><mi>S</mi></mrow><mrow><mi>i</mi><mi>a</mi></mrow></msub></mrow><mo>-</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><msubsup><mo>∑</mo><mrow><mi>a</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><mrow><msub><mrow><mi>S</mi></mrow><mrow><mi>a</mi><mi>j</mi></mrow></msub></mrow><mo>+</mo><mfrac><mn>1</mn><mrow><msup><mrow><mi>N</mi></mrow><mn>2</mn></msup></mrow></mfrac><msubsup><mo>∑</mo><mrow><mi>a</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><mrow><msubsup><mo>∑</mo><mrow><mi>b</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><mrow><msub><mrow><mi>S</mi></mrow><mrow><mi>a</mi><mi>b</mi></mrow></msub></mrow></mrow><mo>.</mo></mrow></mrow></math></script></div></div><p id="__p12">Then, the eigenvalue decomposition of <strong><em><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="MathJax-Span-189" role="math" style="width: 0.747em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px; font-size: 126%;"><span style="position: absolute; clip: rect(-0.097em, 1000.6em, 1.143em, -999.998em); top: -0.99em; left: 0.002em;"><span class="mrow" id="MathJax-Span-190"><span class="mover" id="MathJax-Span-191"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px;"><span style="position: absolute; clip: rect(3.227em, 1000.5em, 4.169em, -999.998em); top: -4.015em; left: 0.052em;"><span class="mi" id="MathJax-Span-192" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span><span style="position: absolute; clip: rect(3.127em, 1000.55em, 3.574em, -999.998em); top: -4.214em; left: 0.052em;"><span class="mo" id="MathJax-Span-193" style=""><span style="font-family: STIXSizeOneSym;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.995em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.316em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-5"><math><mover accent="true"><mi>S</mi><mo>̃</mo></mover></math></script></em></strong> is performed, thereby yielding eigenvalue vector <strong><em>λ</em></strong> and a matrix of eigen-vector <strong><em>Z</em></strong>. Using <strong><em>λ</em></strong> and <strong><em>Z</em></strong>, the coordinates of triangle types are then found using the following formula:</p><div class="disp-formula" id="fd4-13_139"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style=""><nobr><span class="math" id="m4" overflow="scroll" role="math" style="width: 20.214em; display: inline-block;"><span style="display: inline-block; position: relative; width: 16.283em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(1.565em, 1016.23em, 3.128em, -999.997em); top: -2.568em; left: 0.003em;"><span class="mrow" id="MathJax-Span-195"><span class="mrow" id="MathJax-Span-196"><span class="mrow" id="MathJax-Span-197"><span class="mi" id="MathJax-Span-198" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span><span class="mo" id="MathJax-Span-199" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">=</span><span class="mo" id="MathJax-Span-200" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">(</span><span class="msqrt" id="MathJax-Span-201"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.91em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-202"><span class="mrow" id="MathJax-Span-203"><span class="msub" id="MathJax-Span-204"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-205"><span class="mi" id="MathJax-Span-206" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mn" id="MathJax-Span-207" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.91em, 3.38em, -999.997em); top: -3.979em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.406em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-208"><span style="display: inline-block; position: relative; width: 0.809em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-209"><span class="mi" id="MathJax-Span-210" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mn" id="MathJax-Span-211" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-212" style="font-family: STIXGeneral-Regular;">,</span><span class="msqrt" id="MathJax-Span-213" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.91em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-214"><span class="mrow" id="MathJax-Span-215"><span class="msub" id="MathJax-Span-216"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-217"><span class="mi" id="MathJax-Span-218" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mn" id="MathJax-Span-219" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.91em, 3.38em, -999.997em); top: -3.979em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.406em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-220"><span style="display: inline-block; position: relative; width: 0.809em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-221"><span class="mi" id="MathJax-Span-222" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mn" id="MathJax-Span-223" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-224" style="font-family: STIXGeneral-Regular;">,</span><span class="msqrt" id="MathJax-Span-225" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.91em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-226"><span class="mrow" id="MathJax-Span-227"><span class="msub" id="MathJax-Span-228"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-229"><span class="mi" id="MathJax-Span-230" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mn" id="MathJax-Span-231" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.91em, 3.38em, -999.997em); top: -3.979em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.406em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-232"><span style="display: inline-block; position: relative; width: 0.809em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-233"><span class="mi" id="MathJax-Span-234" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mn" id="MathJax-Span-235" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-236" style="font-family: STIXGeneral-Regular;">,</span><span class="mo" id="MathJax-Span-237" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">…</span><span class="mo" id="MathJax-Span-238" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">,</span><span class="msqrt" id="MathJax-Span-239" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.666em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-240"><span class="mrow" id="MathJax-Span-241"><span class="msub" id="MathJax-Span-242"><span style="display: inline-block; position: relative; width: 0.708em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-243"><span class="mi" id="MathJax-Span-244" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mi" id="MathJax-Span-245" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">l<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.76em, 3.38em, -999.997em); top: -3.979em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.759em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.255em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-246"><span style="display: inline-block; position: relative; width: 0.658em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-247"><span class="mi" id="MathJax-Span-248" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mi" id="MathJax-Span-249" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">l<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-250" style="font-family: STIXGeneral-Regular;">)</span><span class="mo" id="MathJax-Span-251" style="font-family: STIXGeneral-Regular;">.</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.573em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.559em; border-left: 0px solid; width: 0px; height: 1.691em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="m4" overflow="scroll"><mrow><mrow><mi mathvariant="bold-italic">X</mi><mo>=</mo><mo stretchy="false">(</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mn>1</mn></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mn>1</mn></msub><mo>,</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mn>2</mn></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mn>2</mn></msub><mo>,</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mn>3</mn></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mn>3</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mi>l</mi></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mi>l</mi></msub><mo stretchy="false">)</mo><mo>.</mo></mrow></mrow></math></script></div></div><p id="__p13" class="p p-last">We used the randomized algorithm [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b12-13_139" rid="b12-13_139" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">12</a>] to conduct large-scale singular-value decomposition for eigenvalue decomposition of <strong><em><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-7-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="MathJax-Span-252" role="math" style="width: 0.747em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px; font-size: 126%;"><span style="position: absolute; clip: rect(-0.097em, 1000.6em, 1.143em, -999.998em); top: -0.99em; left: 0.002em;"><span class="mrow" id="MathJax-Span-253"><span class="mover" id="MathJax-Span-254"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px;"><span style="position: absolute; clip: rect(3.227em, 1000.5em, 4.169em, -999.998em); top: -4.015em; left: 0.052em;"><span class="mi" id="MathJax-Span-255" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span><span style="position: absolute; clip: rect(3.127em, 1000.55em, 3.574em, -999.998em); top: -4.214em; left: 0.052em;"><span class="mo" id="MathJax-Span-256" style=""><span style="font-family: STIXSizeOneSym;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.995em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.316em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-7"><math><mover accent="true"><mi>S</mi><mo>̃</mo></mover></math></script></em></strong> because the order of <strong><em><span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-8-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="MathJax-Span-257" role="math" style="width: 0.747em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px; font-size: 126%;"><span style="position: absolute; clip: rect(-0.097em, 1000.6em, 1.143em, -999.998em); top: -0.99em; left: 0.002em;"><span class="mrow" id="MathJax-Span-258"><span class="mover" id="MathJax-Span-259"><span style="display: inline-block; position: relative; width: 0.598em; height: 0px;"><span style="position: absolute; clip: rect(3.227em, 1000.5em, 4.169em, -999.998em); top: -4.015em; left: 0.052em;"><span class="mi" id="MathJax-Span-260" style="font-family: STIXGeneral-Italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span><span style="position: absolute; clip: rect(3.127em, 1000.55em, 3.574em, -999.998em); top: -4.214em; left: 0.052em;"><span class="mo" id="MathJax-Span-261" style=""><span style="font-family: STIXSizeOneSym;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.02em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 0.995em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.059em; border-left: 0px solid; width: 0px; height: 1.316em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-8"><math><mover accent="true"><mi>S</mi><mo>̃</mo></mover></math></script></em></strong> is huge, and because the only necessary eigenvalues are those with a large absolute value.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Convert pockets into reduced vector representations</h3><p id="__p14" class="p p-first">We defined <strong><em>n</em></strong> as a 295,240-dimensional vector based on the occurrence frequencies of triangle types at a ligand-binding pocket. All triangles that occur in a pocket with edge lengths of 1.0 Å to 15.8 Å are classified as one of 295,240 triangle types. Using <strong><em>X</em></strong> described in the previous section, we found the following.</p><div class="disp-formula" id="fd5-13_139"><div class="f"><span class="MathJax_Preview"></span><div class="MathJax_Display" style="text-align: center;"><span class="MathJax" id="MathJax-Element-9-Frame" tabindex="0" style=""><nobr><span class="math" id="m5" overflow="scroll" role="math" style="width: 34.277em; display: inline-block;"><span style="display: inline-block; position: relative; width: 27.623em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(-0.502em, 1027.47em, 4.337em, -999.997em); top: -2.165em; left: 0.003em;"><span class="mrow" id="MathJax-Span-263"><span class="mrow" id="MathJax-Span-264"><span class="mtable" id="MathJax-Span-265" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 27.321em; height: 0px;"><span style="position: absolute; clip: rect(2.876em, 1027.32em, 7.714em, -999.997em); top: -5.542em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 27.321em; height: 0px;"><span style="position: absolute; clip: rect(2.724em, 1027.32em, 4.488em, -999.997em); top: -5.391em; left: 0.003em;"><span class="mtd" id="MathJax-Span-266"><span class="mrow" id="MathJax-Span-267"><span class="msup" id="MathJax-Span-268"><span style="display: inline-block; position: relative; width: 1.263em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-269"><span class="mi" id="MathJax-Span-270" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.053em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.383em; left: 0.759em;"><span class="mtext" id="MathJax-Span-271" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mi" id="MathJax-Span-272" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">n</span><span class="mo" id="MathJax-Span-273" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">=</span><span class="msup" id="MathJax-Span-274" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 14.519em; height: 0px;"><span style="position: absolute; clip: rect(2.926em, 1013.96em, 4.488em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-275"><span class="mo" id="MathJax-Span-276" style="font-family: STIXGeneral-Regular;">(</span><span class="msqrt" id="MathJax-Span-277"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.91em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-278"><span class="mrow" id="MathJax-Span-279"><span class="msub" id="MathJax-Span-280"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-281"><span class="mi" id="MathJax-Span-282" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mn" id="MathJax-Span-283" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.91em, 3.38em, -999.997em); top: -4.03em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.406em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-284"><span style="display: inline-block; position: relative; width: 0.809em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-285"><span class="mi" id="MathJax-Span-286" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mn" id="MathJax-Span-287" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-288" style="font-family: STIXGeneral-Regular;">,</span><span class="msqrt" id="MathJax-Span-289" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.91em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-290"><span class="mrow" id="MathJax-Span-291"><span class="msub" id="MathJax-Span-292"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-293"><span class="mi" id="MathJax-Span-294" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mn" id="MathJax-Span-295" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.91em, 3.38em, -999.997em); top: -4.03em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.406em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-296"><span style="display: inline-block; position: relative; width: 0.809em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-297"><span class="mi" id="MathJax-Span-298" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mn" id="MathJax-Span-299" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-300" style="font-family: STIXGeneral-Regular;">,</span><span class="msqrt" id="MathJax-Span-301" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.817em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.91em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-302"><span class="mrow" id="MathJax-Span-303"><span class="msub" id="MathJax-Span-304"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-305"><span class="mi" id="MathJax-Span-306" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mn" id="MathJax-Span-307" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.91em, 3.38em, -999.997em); top: -4.03em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.406em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-308"><span style="display: inline-block; position: relative; width: 0.809em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-309"><span class="mi" id="MathJax-Span-310" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mn" id="MathJax-Span-311" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-312" style="font-family: STIXGeneral-Regular;">,</span><span class="mo" id="MathJax-Span-313" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">…</span><span class="mo" id="MathJax-Span-314" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">,</span><span class="msqrt" id="MathJax-Span-315" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.666em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.71em, 4.287em, -999.997em); top: -3.979em; left: 0.91em;"><span class="mrow" id="MathJax-Span-316"><span class="mrow" id="MathJax-Span-317"><span class="msub" id="MathJax-Span-318"><span style="display: inline-block; position: relative; width: 0.708em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-319"><span class="mi" id="MathJax-Span-320" style="font-family: STIXGeneral-Italic;">λ</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.456em;"><span class="mi" id="MathJax-Span-321" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">l<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.027em, 1000.76em, 3.38em, -999.997em); top: -4.03em; left: 0.91em;"><span style="display: inline-block; position: relative; width: 0.759em; height: 0px;"><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.003em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; font-family: STIXGeneral-Regular; top: -3.979em; left: 0.255em;">‾<span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.876em, 1000.96em, 4.388em, -999.997em); top: -3.879em; left: 0.003em;"><span style="font-family: STIXGeneral-Regular;">√</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msub" id="MathJax-Span-322"><span style="display: inline-block; position: relative; width: 0.658em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.36em, 4.236em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-323"><span class="mi" id="MathJax-Span-324" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">z</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.406em;"><span class="mi" id="MathJax-Span-325" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">l<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-326" style="font-family: STIXGeneral-Regular;">)</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.584em; left: 14.015em;"><span class="mtext" id="MathJax-Span-327" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="msup" id="MathJax-Span-328"><span style="display: inline-block; position: relative; width: 9.68em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1009.13em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-329"><span class="mo" id="MathJax-Span-330" style="font-family: STIXGeneral-Regular;">(</span><span class="msub" id="MathJax-Span-331"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-332"><span class="mi" id="MathJax-Span-333" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-334" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-335" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-336" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-337"><span class="mi" id="MathJax-Span-338" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-339" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-340" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-341" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-342"><span class="mi" id="MathJax-Span-343" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-344" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-345" style="font-family: STIXGeneral-Regular;">,</span><span class="mo" id="MathJax-Span-346" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">…</span><span class="mo" id="MathJax-Span-347" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">,</span><span class="msub" id="MathJax-Span-348" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 2.674em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-349"><span class="mi" id="MathJax-Span-350" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-351"><span class="mn" id="MathJax-Span-352" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">295240</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-353" style="font-family: STIXGeneral-Regular;">)</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.383em; left: 9.176em;"><span class="mtext" id="MathJax-Span-354" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.976em, 1019.81em, 4.337em, -999.997em); top: -3.727em; left: 0.003em;"><span class="mtd" id="MathJax-Span-355"><span class="mrow" id="MathJax-Span-356"><span class="mo" id="MathJax-Span-357" style="font-family: STIXGeneral-Regular;">=</span><span class="mo" id="MathJax-Span-358" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">(</span><span class="msub" id="MathJax-Span-359"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-360"><span class="mi" id="MathJax-Span-361" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">x</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-362" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-363" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-364" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-365"><span class="mi" id="MathJax-Span-366" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">x</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-367" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-368" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-369" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-370"><span class="mi" id="MathJax-Span-371" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">x</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-372" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-373" style="font-family: STIXGeneral-Regular;">,</span><span class="mo" id="MathJax-Span-374" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">…</span><span class="mo" id="MathJax-Span-375" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">,</span><span class="msub" id="MathJax-Span-376" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 2.674em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-377"><span class="mi" id="MathJax-Span-378" style="font-family: STIXGeneral; font-style: italic; font-weight: bold;">x</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-379"><span class="mn" id="MathJax-Span-380" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">295240</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-381" style="font-family: STIXGeneral-Regular;">)</span><span class="msup" id="MathJax-Span-382"><span style="display: inline-block; position: relative; width: 9.68em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1009.13em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-383"><span class="mo" id="MathJax-Span-384" style="font-family: STIXGeneral-Regular;">(</span><span class="msub" id="MathJax-Span-385"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-386"><span class="mi" id="MathJax-Span-387" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-388" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-389" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-390" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-391"><span class="mi" id="MathJax-Span-392" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-393" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-394" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-395" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-396"><span class="mi" id="MathJax-Span-397" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mn" id="MathJax-Span-398" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-399" style="font-family: STIXGeneral-Regular;">,</span><span class="mo" id="MathJax-Span-400" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">…</span><span class="mo" id="MathJax-Span-401" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">,</span><span class="msub" id="MathJax-Span-402" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 2.674em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.46em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-403"><span class="mi" id="MathJax-Span-404" style="font-family: STIXGeneral-Italic;">n</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.507em;"><span class="mrow" id="MathJax-Span-405"><span class="mn" id="MathJax-Span-406" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">295240</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-407" style="font-family: STIXGeneral-Regular;">)</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.383em; left: 9.176em;"><span class="mtext" id="MathJax-Span-408" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(2.976em, 1011.34em, 4.337em, -999.997em); top: -2.165em; left: 0.003em;"><span class="mtd" id="MathJax-Span-409"><span class="mrow" id="MathJax-Span-410"><span class="mo" id="MathJax-Span-411" style="font-family: STIXGeneral-Regular;">=</span><span class="msup" id="MathJax-Span-412" style="padding-left: 0.305em;"><span style="display: inline-block; position: relative; width: 8.42em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1007.87em, 4.337em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-413"><span class="mo" id="MathJax-Span-414" style="font-family: STIXGeneral-Regular;">(</span><span class="msub" id="MathJax-Span-415"><span style="display: inline-block; position: relative; width: 1.111em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.66em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-416"><span class="mi" id="MathJax-Span-417" style="font-family: STIXGeneral-Italic;">w</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.658em;"><span class="mn" id="MathJax-Span-418" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">1</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-419" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-420" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.111em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.66em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-421"><span class="mi" id="MathJax-Span-422" style="font-family: STIXGeneral-Italic;">w</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.658em;"><span class="mn" id="MathJax-Span-423" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">2</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-424" style="font-family: STIXGeneral-Regular;">,</span><span class="msub" id="MathJax-Span-425" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 1.111em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.66em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-426"><span class="mi" id="MathJax-Span-427" style="font-family: STIXGeneral-Italic;">w</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.658em;"><span class="mn" id="MathJax-Span-428" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">3</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-429" style="font-family: STIXGeneral-Regular;">,</span><span class="mo" id="MathJax-Span-430" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">…</span><span class="mo" id="MathJax-Span-431" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">,</span><span class="msub" id="MathJax-Span-432" style="padding-left: 0.204em;"><span style="display: inline-block; position: relative; width: 0.96em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.66em, 4.136em, -999.997em); top: -3.979em; left: 0.003em;"><span class="mrow" id="MathJax-Span-433"><span class="mi" id="MathJax-Span-434" style="font-family: STIXGeneral-Italic;">w</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -3.828em; left: 0.658em;"><span class="mi" id="MathJax-Span-435" style="font-size: 70.7%; font-family: STIXGeneral-Italic;">l<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-436" style="font-family: STIXGeneral-Regular;">)</span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; top: -4.383em; left: 7.916em;"><span class="mtext" id="MathJax-Span-437" style="font-size: 70.7%; font-family: STIXGeneral-Regular;">T</span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-438" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">=</span><span class="mi" id="MathJax-Span-439" style="font-family: STIXGeneral; font-style: italic; font-weight: bold; padding-left: 0.305em;">w</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 5.547em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.17em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -2.559em; border-left: 0px solid; width: 0px; height: 5.753em;"></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-9"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="m5" overflow="scroll"><mrow><mtable columnalign="left" columnspacing="2pt"><mtr><mtd><msup><mrow><mi mathvariant="bold-italic">X</mi></mrow><mtext>T</mtext></msup><mi mathvariant="bold-italic">n</mi><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mn>1</mn></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mn>1</mn></msub><mo>,</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mn>2</mn></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mn>2</mn></msub><mo>,</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mn>3</mn></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mn>3</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msqrt><mrow><msub><mrow><mi>λ</mi></mrow><mi>l</mi></msub></mrow></msqrt><msub><mrow><mi mathvariant="bold-italic">z</mi></mrow><mi>l</mi></msub><mo stretchy="false">)</mo></mrow><mtext>T</mtext></msup><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mi>n</mi></mrow><mn>1</mn></msub><mo>,</mo><msub><mrow><mi>n</mi></mrow><mn>2</mn></msub><mo>,</mo><msub><mrow><mi>n</mi></mrow><mn>3</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>295240</mn></mrow></msub><mo stretchy="false">)</mo></mrow><mtext>T</mtext></msup></mtd></mtr><mtr><mtd><mo>=</mo><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="bold-italic">x</mi></mrow><mn>1</mn></msub><mo>,</mo><msub><mrow><mi mathvariant="bold-italic">x</mi></mrow><mn>2</mn></msub><mo>,</mo><msub><mrow><mi mathvariant="bold-italic">x</mi></mrow><mn>3</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi mathvariant="bold-italic">x</mi></mrow><mrow><mn>295240</mn></mrow></msub><mo stretchy="false">)</mo><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mi>n</mi></mrow><mn>1</mn></msub><mo>,</mo><msub><mrow><mi>n</mi></mrow><mn>2</mn></msub><mo>,</mo><msub><mrow><mi>n</mi></mrow><mn>3</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>295240</mn></mrow></msub><mo stretchy="false">)</mo></mrow><mtext>T</mtext></msup></mtd></mtr><mtr><mtd><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mi>w</mi></mrow><mn>1</mn></msub><mo>,</mo><msub><mrow><mi>w</mi></mrow><mn>2</mn></msub><mo>,</mo><msub><mrow><mi>w</mi></mrow><mn>3</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>w</mi></mrow><mi>l</mi></msub><mo stretchy="false">)</mo></mrow><mtext>T</mtext></msup><mo>=</mo><mi mathvariant="bold-italic">w</mi></mtd></mtr></mtable></mrow></math></script></div></div><p id="__p15" class="p p-last">In the equations above, <em>n</em><em><sub>i</sub></em> represents the number of the <em>i</em>-th triangle in the list of triangle types. <strong><em>w</em></strong> stands for a vector representing a pocket (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f1-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1-13_139" rid-ob="ob-f1-13_139" co-legend-rid="lgnd_f1-13_139"><span>Fig. 1</span></a>). To represent a pocket with a reduced vector based on the MDS result, we used the number of dimensions that satisfy a certain extent of cumulative contribution ratio calculated using only positive eigenvalues (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#s1-13_139" rid="s1-13_139" class=" supplementary-material">Supplementary Fig. S1</a>). For this study, we set the criteria of the cumulative contribution ratio as 0.98. We define similarity between two pockets <em>i</em> and <em>j</em> as a cosine distance between <strong><em>w</em></strong><em><sub>i</sub></em> and <strong><em>w</em></strong><em><sub>j</sub></em>. Therefore, the similarity can be found easily by calculating the inner product between normalized <strong><em>w</em></strong><em><sub>i</sub></em> and normalized <strong><em>w</em></strong><em><sub>j</sub></em>. This procedure can be regarded as calculation of the weighted arithmetic mean over <strong><em>X</em></strong> weighted by <strong><em>n</em></strong>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f1-13_139" co-legend-rid="lgnd_f1-13_139"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f1-13_139/" target="figure" rid-figpopup="f1-13_139" rid-ob="ob-f1-13_139"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140309106890912" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f1-13_139/" target="figure" rid-figpopup="f1-13_139" rid-ob="ob-f1-13_139"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5042158_13_139_f1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is 13_139_f1.jpg" title="Click on image to zoom" class="tileshop" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/13_139_f1.jpg"></a></div><div id="largeobj_idm140309106890912" class="largeobj-link align_right" style=""><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f1-13_139/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f1-13_139"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f1-13_139/" target="figure" rid-figpopup="f1-13_139" rid-ob="ob-f1-13_139">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p41">Schematic diagram of a vector representation of a ligand-binding pocket. Structural and amino acid information of a ligand-binding pocket are converted into a vector based on the MDS result.</p></div></div></div></div><div id="__sec7" class="sec sec-last"><h3 id="__sec7title">Datasets</h3><p id="__p16" class="p p-first">To optimize the weighting factor <em>r</em> in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#fd1-13_139" rid="fd1-13_139" class=" disp-formula">eq. (1)</a> and also to analyze the performance of the methods described above, we used the two datasets used in our previous study [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b8-13_139" rid="b8-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930658" role="button" aria-expanded="false" aria-haspopup="true">8</a>]. One is <em>Ito138.</em> This difficult dataset comprises 138 known ligand-binding pockets. The dataset comprises pocket pairs that share the same types of small molecules in proteins with different global structures. The other is <em>APocS3</em> used in the study of <em>APoc</em> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b5-13_139" rid="b5-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930655" role="button" aria-expanded="false" aria-haspopup="true">5</a>]. While this easy dataset originally comprised 38,066 pairs each in Subject and Control dataset, nevertheless, we noted that some of binding pocket are inadequate because of small number of binding residues, based on the default setting of <em>APoc</em> which is that “minimal number of pocket residue is 10”. We omitted pocket pairs which could not be handled by <em>APoc</em> with default setting, and then confirmed that we reproduced the same ROC curves (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139" co-legend-rid="lgnd_f3-13_139"><span>Fig. 3B</span></a> in [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b5-13_139" rid="b5-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930647" role="button" aria-expanded="false" aria-haspopup="true">5</a>]). For that reason, we used 37,956/26,527 pairs for Subject/Control dataset in this study. Coordinate files of binding pockets were obtained from the <em>APoc</em> website (<a href="http://cssb.biology.gatech.edu/APoc" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cssb.biology.gatech.edu/APoc</a>). We conducted all-against-all 
<span class="MathJax_Preview"></span><span class="MathJax" id="MathJax-Element-10-Frame" tabindex="0" style="font-size: 78%;"><nobr><span class="math" id="m6" overflow="scroll" role="math" style="width: 8.369em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.757em; height: 0px; font-size: 124%;"><span style="position: absolute; clip: rect(0.859em, 1006.71em, 3.682em, -999.997em); top: -2.518em; left: 0.003em;"><span class="mrow" id="MathJax-Span-441"><span class="mrow" id="MathJax-Span-442"><span class="mrow" id="MathJax-Span-443"><span class="mn" id="MathJax-Span-444" style="font-family: STIXGeneral-Regular;">9</span><span class="mo" id="MathJax-Span-445" style="font-family: STIXGeneral-Regular;">,</span><span class="mn" id="MathJax-Span-446" style="font-family: STIXGeneral-Regular; padding-left: 0.204em;">453</span><span class="mi" id="MathJax-Span-447" style="font-family: STIXGeneral-Regular;"><span style="margin-left: 0.154em;"></span></span><span class="mo" id="MathJax-Span-448" style="font-family: STIXGeneral-Regular;">(</span><span class="mo" id="MathJax-Span-449" style="font-family: STIXGeneral-Regular;">=</span><span class="mo" id="MathJax-Span-450" style="font-family: STIXGeneral-Regular; padding-left: 0.305em;">(</span><span class="mtable" id="MathJax-Span-451" style="padding-right: 0.154em; padding-left: 0.154em;"><span style="display: inline-block; position: relative; width: 1.515em; height: 0px;"><span style="position: absolute; clip: rect(2.472em, 1001.46em, 4.942em, -999.997em); top: -3.979em; left: 0.003em;"><span style="display: inline-block; position: relative; width: 1.515em; height: 0px;"><span style="position: absolute; clip: rect(3.178em, 1001.46em, 4.136em, -999.997em); top: -4.685em; left: 50%; margin-left: -0.754em;"><span class="mtd" id="MathJax-Span-452"><span class="mrow" id="MathJax-Span-453"><span class="mrow" id="MathJax-Span-454"><span class="mn" id="MathJax-Span-455" style="font-family: STIXGeneral-Regular;">138</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span><span style="position: absolute; clip: rect(3.178em, 1000.46em, 4.136em, -999.997em); top: -3.173em; left: 50%; margin-left: -0.249em;"><span class="mtd" id="MathJax-Span-456"><span class="mrow" id="MathJax-Span-457"><span class="mn" id="MathJax-Span-458" style="font-family: STIXGeneral-Regular;">2</span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.984em;"></span></span></span></span><span class="mo" id="MathJax-Span-459" style="font-family: STIXGeneral-Regular;">)</span><span class="mo" id="MathJax-Span-460" style="font-family: STIXGeneral-Regular;">)</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.523em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.309em; border-left: 0px solid; width: 0px; height: 3.191em;"></span></span></nobr></span><script type="math/mml" id="MathJax-Element-10"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="m6" overflow="scroll"><mrow><mrow><mn>9</mn><mo>,</mo><mn>453</mn><mi> </mi><mo stretchy="false">(</mo><mo>=</mo><mo stretchy="false">(</mo><mtable><mtr><mtd><mrow><mn>138</mn></mrow></mtd></mtr><mtr><mtd><mn>2</mn></mtd></mtr></mtable><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></mrow></math></script> comparisons for the <em>Ito138</em> dataset. For <em>APocS3</em>, we performed comparisons of the 64,483 ligand-binding pocket pairs for the Subject and Control datasets. We regarded ligand-binding pocket pairs that share the same (for <em>Ito138</em> &amp; <em>APocS3</em>) and also similar (for <em>APocS3</em>) ligands as relevant pairs (Subject); otherwise we regarded pocket pairs as non-relevant pairs (Control). For a pocket pair with the same ligand, i.e., a relevant pair with a higher similarity score than a threshold value, it was regarded as a true positive (TP). Otherwise, it was regarded as a false negative (FN). If a pair with ligands that are not the same, i.e., non-relevant pair has a lower similarity score than the threshold value, then it was regarded as a true negative (TN). Otherwise, it was regarded as a false positive (FP). Here, a true positive rate (TPR), also designated as recall or sensitivity, is defined as TP/(TP+FN). A false positive rate (FPR), also designated as fall-out, is given as FP/(FP+TN). Then, the receiver operating characteristic (ROC) curve is used to present these results. Actually, ROC is a curve based on TPR against FPR at various thresholds. The area under the ROC curve (AUC) is used for performance evaluation of the methods.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f3-13_139" co-legend-rid="lgnd_f3-13_139"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140309110310112" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5042158_13_139_f3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is 13_139_f3.jpg" title="Click on image to zoom" class="tileshop" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/13_139_f3.jpg"></a></div><div id="largeobj_idm140309110310112" class="largeobj-link align_right" style="display: none;"><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f3-13_139"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p43">Benchmark results with <em>APocS3</em>. (a) AUC values of our new methods, including ‘5 edge’, ‘6 edge α’, and ‘6 edge β’, with various values (0.05–0.95 with the sampling interval of 0.05) of the weighting factor <em>r</em> are shown. (b) ROC curves of our new methods, our previous dissimilarity-based method (<em>r</em>=0.35), and <em>APoc</em> are shown.</p></div></div></div><p id="__p17" class="p p-last">Additionally, we used the third dataset, which we call <em>APocS3_LIGSITE</em>, to compare our new method with <em>APoc</em>. This dataset included pairs of predicted pockets generated by LIGSITE [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b13-13_139" rid="b13-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930663" role="button" aria-expanded="false" aria-haspopup="true">13</a>] based on pocket pairs in <em>APocS3.</em> In comparison with <em>APocS3</em>, pocket pairs were reduced to 34,511/17,408 pairs of the Subject/Control dataset because binding residues of some pockets could not be predicted correctly.</p></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Results and Discussion</h2><p id="__p18" class="p p-first">We investigated the effects of modifications in the following three points, i.e., the new similarity definition, the expansion of edge classes, and the revision of intervals of edge classes. Then we compared our new method with an existing fast sequence order-independent structural alignment method: <em>APoc</em>.</p><div id="__sec9" class="sec"><h3 id="__sec9title">Effects of the new similarity definition between pockets</h3><p id="__p19" class="p p-first">First, we evaluated the effectiveness of changing the similarity definition between two ligand-binding pockets. For evaluation, a new method was used with the number of edge classes set as five classes. The method of classifying them is the same as that of our previous method. We tested the weighting factor in every 0.05 sampling from 0.05 to 0.95 to define the optimized value of <em>r</em> using the <em>Ito138</em> dataset. Plots of the weighting factor vs. AUC are presented in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139" co-legend-rid="lgnd_f2-13_139"><span>Figure 2a</span></a> as ‘5 edge’. According to this result, the best AUC is obtained with <em>r</em>=0.15. Therefore, we used this value to evaluate the effectiveness of the new similarity definition. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139" co-legend-rid="lgnd_f2-13_139"><span>Figure 2b</span></a> presents ROC curves, i.e., plots of TPR vs. FPR for this evaluation and shows that the new similarity definition outperforms the previous dissimilarity definition. We identified the main reason behind the superiority of the new definition. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139" co-legend-rid="lgnd_f2-13_139"><span>Figure 2c</span></a> presents actual similarity values for all-against-all 9,453 pairs as a heat map to compare the new similarity-based method (shown at the lower left) with our previous method (shown at the upper right). Each square in the graph corresponds to one similarity of a pair of pockets. Relative reddish/blueish in the color scale in a method is important. We found that the new method assigns lower similarity to pockets, especially to those which bind to HEM or SF4 with pockets which bind to the other ligands.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f2-13_139" co-legend-rid="lgnd_f2-13_139"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140309187365536" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5042158_13_139_f2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is 13_139_f2.jpg" title="Click on image to zoom" class="tileshop" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/13_139_f2.jpg"></a></div><div id="largeobj_idm140309187365536" class="largeobj-link align_right" style=""><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f2-13_139"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p42">Benchmark results with <em>Ito138</em>. (a) Results of our new methods, including ‘5 edge’, ‘6 edge (with interval set) α’, and ‘6 edge (with interval set) β’, with various values (0.05–0.95 with the sampling interval of 0.05) of the weighting factor <em>r</em> are shown. The X axis indicates <em>r</em>, and the Y axis indicates AUC values. (b) ROC curves of our new methods and our previous dissimilarity-based method (<em>r</em>=0.35) are shown. The X axis shows FPR. The Y axis shows TPR. (c) Heat maps to compare the new (5 edge class) similarity-based method (lower left) with the dissimilarity-based method (upper right) are shown. The color of each square in the map represents a similarity value for a pocket pair. Ligand abbreviations placed by axes correspond to the ligand to which a pocket binds. (d) Heat maps to compare the ‘6 edge α’ method (lower left) with the ‘5 edge class’ method (upper right) are shown.</p></div></div></div><p id="__p20">Next, we evaluated the effectiveness using <em>APocS3</em>. We tested weighting factor <em>r</em> in the manner described above. Plots of the weighting factor vs. AUC are presented in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139" co-legend-rid="lgnd_f3-13_139"><span>Figure 3a</span></a> as ‘5 edge’. According to this result, the best AUC is given by 0.10. Therefore, we used it to evaluate the effectiveness. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139" co-legend-rid="lgnd_f3-13_139"><span>Figure 3b</span></a> presents ROC curves for this evaluation. It shows that the new similarity definition also outperforms previous dissimilarity definition. We considered that the parameter which gives the best AUC changed from 0.15 to 0.10 because the <em>APocS3</em> dataset is more conserved for pocket shape than <em>Ito138</em>. Therefore, the geometrical similarity term in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#fd1-13_139" rid="fd1-13_139" class=" disp-formula">eq. (1)</a> is more weighted to distinguish a subject pair from a control pair.</p><p id="__p21" class="p p-last">Furthermore, to investigate the effects of similarity definitions, we compared the results obtained by combining previous/new definitions and different number of dimensions used in vector representation: i) the previous dissimilarity definition and new 139/137 dimensions, ii) the new similarity definition and previous 11 dimensions, and iii) the new similarity definition and the same criteria used in our previous study [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b8-13_139" rid="b8-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930654" role="button" aria-expanded="false" aria-haspopup="true">8</a>] for deciding the number of dimensions. Here, the criteria used in the previous study is “only use positive eigenvalues which are greater than the absolute value of negative minimum eigenvalue”. These results are shown in <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#s1-13_139" rid="s1-13_139" class=" supplementary-material">Supplementary Table S1</a>. When the same criteria used in the previous study was employed, the results with similarity are superior to ones with dissimilarity. Indeed, the results with similarity are superior to ones with dissimilarity except “11 dimensions (fixed)” in <em>APocS3</em>. We concluded that the improvements described above largely depends on the new similarity measure. We speculated that these improvements came from the difference of the matrix used for deriving the dissimilarity/similarity matrix. The Miyata’s matrix which was used previously is just created from physicochemical characteristics in amino acids. On the contrary, the PAM50 matrix which was used for new measure reflects the evolutions of proteins and the concept based on our assumption that residues of pockets are less mutative.</p></div><div id="__sec10" class="sec"><h3 id="__sec10title">Effects of increasing the number of edge classes</h3><p id="__p22" class="p p-first">Next, we increased the number of edge classes from 5 to 6, according to the expansion of Cα–Cα distances of residue pairs, ranging from 1.0 Å to 15.8 Å, used as the triangle edge. Edges were classified into 6 classes at intervals of 2.2 Å (interval set α). First the effects of this modification were evaluated using <em>Ito138</em>. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139" co-legend-rid="lgnd_f2-13_139"><span>Figures 2a</span></a>, b show that 6 edge classes outperform 5 edge classes, which suggests that addition of edge classes engenders better ability to recognize similar binding pockets. In <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139" co-legend-rid="lgnd_f2-13_139"><span>Figure 2d</span></a>, as heat maps, we compared individual result obtained using the ‘6 edge α’ method (shown in the lower left) with it using the ‘5 edge class’ method (shown in the upper right). In this case we also found that discrimination of HEM binding pockets from SF4 binding pockets is improved in the ‘6 edge α’ method compared with the ‘5 edge class’ method. Squares which correspond to HEM binding pockets vs SF4 binding pockets are less reddish/orangish in ‘6 edge alpha class’ than ‘5 edge class’, and boxes which correspond to HEM binding pockets vs HEM binding pockets are more reddish. Similarly, the discriminate power of GDP binding pockets from other pockets by the ‘6 edge α’ method is slightly better than that of the ‘5 edge class’ method. We suppose that discrimination of HEM binding pockets and SF4 binding pockets became better because the maximum value of edge length changed to 15.8 Å. The HEM binding pocket is commonly large. The distance between some binding residues which face each other through HEM is about 15 Å.</p><p id="__p23" class="p p-last">We also evaluated the effectiveness of our new method using an easy dataset: <em>APocS3</em>. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139" co-legend-rid="lgnd_f3-13_139"><span>Figures 3a</span></a>, b show that the ‘6 edge α’ method also outperforms 5 edge method. Similar but slightly different results were found with the results described above obtained with <em>Ito138</em>. Regarding AUC values along with the weighting factor <em>r</em> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139" co-legend-rid="lgnd_f3-13_139"><span>Fig. 3a</span></a>), the ‘6 edge α’ method is better than the 5 edge method, except for the range of <em>r</em>=0.05, and 0.50 and more. With the new similarity definition, both methods showed maximum AUC values at <em>r</em>=0.10.</p></div><div id="__sec11" class="sec"><h3 id="__sec11title">Effects of the intervals of class of edge labels</h3><p id="__p24" class="p p-first-last">Next, we examined the procedure used to decide the intervals and modified the intervals of 6 edge classes from 2.2 Å each to 1.0, 4.0, 6.36, 8.72, 11.08, 13.44, and 15.8 Å (interval set β). The setting of every 2.2 Å interval is the same as that of our original method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b6-13_139" rid="b6-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930653" role="button" aria-expanded="false" aria-haspopup="true">6</a>]. The first interval was set to 4.8 Å, the distance originated from the FuzCav method [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b14-13_139" rid="b14-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930657" role="button" aria-expanded="false" aria-haspopup="true">14</a>]. However, we investigated the frequency of the edge length of triangles taken from all pockets in the <em>Ito138</em> dataset (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f4-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4-13_139" rid-ob="ob-f4-13_139" co-legend-rid="lgnd_f4-13_139"><span>Fig. 4</span></a>). In the figure, the green line shows the frequencies of all edge lengths. The red line shows the frequency of edge lengths which comprise two adjacent residues in a chain. According to this figure, almost all edges shorter than about 4.0 Å comprise adjacent residues. Thus, we considered it natural to set the first interval as 4.0 Å based on the difference of chemical characteristics between adjacent residues, or lack thereof. First the effectiveness of this modification was evaluated using <em>Ito138</em>. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f2-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-13_139" rid-ob="ob-f2-13_139" co-legend-rid="lgnd_f2-13_139"><span>Figures 2a</span></a>, b show that this modification was not so influential, in terms of AUC values, to our new method on <em>Ito138</em>, probably because <em>Ito138</em> comprises pockets that are too diverse to be affected by this improvement. On the other hand, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f3-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-13_139" rid-ob="ob-f3-13_139" co-legend-rid="lgnd_f3-13_139"><span>Figures 3a</span></a>, b show that the ‘6 edge β’ method is better than that of the ‘6 edge α’ method on <em>APocS3</em>. Moreover, comparing with <em>APoc</em>, in the low-FPR region, <em>APoc</em> showed higher performance. However, in the region higher than 3.7% FPR, the ‘6 edge β’ method showed higher performance than that of <em>APoc</em>. The result that <em>APoc</em> is superior in low FPR region is also shown in our original method PoSSuM, which is also alignment-free method (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#s1-13_139" rid="s1-13_139" class=" supplementary-material">Supplementary Fig. S2</a>). Here, we used the encoding set 3, which showed the highest AUC among the first four sets (1~4), used in PoSSuM for labeling residues in a pocket [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b6-13_139" rid="b6-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930661" role="button" aria-expanded="false" aria-haspopup="true">6</a>]. We found that the ‘6 edge β’ method achieved the highest AUC among the methods. Note that, in this experiment, we used 26,407 pairs out of 26,527 ones of the Control dataset, due to lack of triangles, based on the definition of the encoding set 3, in some pockets (some amino acids are assigned no vertex labels). Thus, it is noteworthy that our new method can handle all pairs because all 20 types of amino acids are assigned vertex labels.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f4-13_139" co-legend-rid="lgnd_f4-13_139"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f4-13_139/" target="figure" rid-figpopup="f4-13_139" rid-ob="ob-f4-13_139"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140309110306784" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f4-13_139/" target="figure" rid-figpopup="f4-13_139" rid-ob="ob-f4-13_139"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5042158_13_139_f4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is 13_139_f4.jpg" title="Click on image to zoom" class="tileshop" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/13_139_f4.jpg"></a></div><div id="largeobj_idm140309110306784" class="largeobj-link align_right" style=""><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f4-13_139/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f4-13_139"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f4-13_139/" target="figure" rid-figpopup="f4-13_139" rid-ob="ob-f4-13_139">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p44">Distribution of the edge length (Cα–Cαdistance) of triangles taken from all pockets in <em>Ito138</em>.</p></div></div></div></div><div id="__sec12" class="sec"><h3 id="__sec12title">Performance comparison with <em>APocS3_LIGSITE</em></h3><p id="__p25" class="p p-first-last">Finally, we compared the ‘6 edge β’ method with <em>APoc</em> using the <em>APocS3_LIGSITE</em> dataset. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f5-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-13_139" rid-ob="ob-f5-13_139" co-legend-rid="lgnd_f5-13_139"><span>Figure 5</span></a> shows that, in the low-FPR region, <em>APoc</em> showed higher performance. However, in the region higher than 17% FPR, the ‘6 edge β’ method showed higher performance than that of <em>APoc</em>. Additionally, in the perspective of AUC, the ‘6 edge β’ method showed higher performance than that of <em>APoc</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f5-13_139" co-legend-rid="lgnd_f5-13_139"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f5-13_139/" target="figure" rid-figpopup="f5-13_139" rid-ob="ob-f5-13_139"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140309112711632" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f5-13_139/" target="figure" rid-figpopup="f5-13_139" rid-ob="ob-f5-13_139"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5042158_13_139_f5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is 13_139_f5.jpg" title="Click on image to zoom" class="tileshop" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/13_139_f5.jpg"></a></div><div id="largeobj_idm140309112711632" class="largeobj-link align_right" style=""><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f5-13_139/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f5-13_139"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f5-13_139/" target="figure" rid-figpopup="f5-13_139" rid-ob="ob-f5-13_139">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p45">Benchmark results with <em>APocS3_LIGSITE</em>. ROC curves of our new method and <em>APoc</em> are shown.</p></div></div></div></div><div id="__sec13" class="sec"><h3 id="__sec13title">Expedient examples of our method</h3><p id="__p26" class="p p-first">We present examples that demonstrate the usefulness of our new method by comparison to <em>APoc</em>. First, we show an example from <em>Ito138</em> dataset. The interferon-inducible p47 resistance GTPases from mouse (PDBID: 1TQ4 [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b15-13_139" rid="b15-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930662" role="button" aria-expanded="false" aria-haspopup="true">15</a>]) and the alpha1,3-fucosyltransferase with GDP from <em>H. pylori</em> (2NZX [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b16-13_139" rid="b16-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930649" role="button" aria-expanded="false" aria-haspopup="true">16</a>]) have the same ligand: GDP, though the two proteins possess different global structures; P-loop containing nucleoside triphosphate hydrolases fold (1TQ4) and UDP-Glycosyltransferase/glycogen phosphorylase fold (2NZX). Our new method gave 0.864 as the similarity score for this pocket pair (the higher the similarity score, the more likely the pair is composed of pockets to which the same/similar ligand bind). It is noteworthy that <em>APoc</em> gave 0.772 as the <em>p</em>-value for this pair (the lower the <em>p</em>-value, the more likely the pair is composed of pockets to which the same/similar ligand bind), and gave 0.307 as the raw “Pocket Similarity score (PS-score)” (the higher the PS-score, the more likely the pair is composed of pockets to which the same/similar ligand bind).</p><p id="__p27" class="p p-last">We present two more examples from the <em>APocS3</em> dataset. The aldehyde dehydrogenase from rat (1AD3 [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b17-13_139" rid="b17-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930667" role="button" aria-expanded="false" aria-haspopup="true">17</a>]; ALDH-like fold) and the 17-beta-hydroxysteroid dehydrogenase type 4 from human (1ZBQ; NAD(P)-binding Rossmann fold) have the same ligand: NAD (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f6-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f6-13_139" rid-ob="ob-f6-13_139" co-legend-rid="lgnd_f6-13_139"><span>Fig. 6(a)</span></a>). As discussed in the <em>Discussion</em> and <em>Conclusion</em> sections in the paper about <em>APoc</em>, this is an example of dissimilar pockets with different ligand conformations. <em>APoc</em> assigned this pair of pockets a <em>p</em>-value of 0.418 (and gave 0.325 as the PS-score), even though our new method produced a similarity score of 0.872. We regard this fact as demonstrating the effect of usefulness of our alignment-free method, which can accommodate the pocket conformation change associated with the ligand conformation change. Furthermore, the asparagine synthetase from <em>E. coli</em> (12AS [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b18-13_139" rid="b18-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930660" role="button" aria-expanded="false" aria-haspopup="true">18</a>]) and the electron transfer flavoprotein from human (1EFV [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b19-13_139" rid="b19-13_139" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_581930651" role="button" aria-expanded="false" aria-haspopup="true">19</a>]) have the same ligand: AMP (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f6-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f6-13_139" rid-ob="ob-f6-13_139" co-legend-rid="lgnd_f6-13_139"><span>Fig. 6(b)</span></a>). Similarly, as discussed in the paper related to <em>APoc</em>, this is an example of dissimilar pockets and similar ligand conformations. Whereas <em>APoc</em> gave 0.212 as a <em>p</em>-value (and gave 0.337 as the PS-score) for this pocket pair, our new method showed a similarity score of 0.731. We regard this feature as demonstrating the usefulness of this alignment-free method, which can vaguely represent the circumstances related to a ligand.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f6-13_139" co-legend-rid="lgnd_f6-13_139"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f6-13_139/" target="figure" rid-figpopup="f6-13_139" rid-ob="ob-f6-13_139"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140309112708608" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f6-13_139/" target="figure" rid-figpopup="f6-13_139" rid-ob="ob-f6-13_139"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5042158_13_139_f6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is 13_139_f6.jpg" title="Click on image to zoom" class="tileshop" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/13_139_f6.jpg"></a></div><div id="largeobj_idm140309112708608" class="largeobj-link align_right" style=""><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f6-13_139/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f6-13_139"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f6-13_139/" target="figure" rid-figpopup="f6-13_139" rid-ob="ob-f6-13_139">Figure 6</a></div><!--caption a7--><div class="caption"><p id="__p46">Superpositions of two expedient examples detected by our method. The superpositions are obtained by minimizing RMSD between the each ligands. (a) 1AD3 (cyan and green) and 1ZBQ (purple and red), which bind the same ligand: NAD. The Ca atoms of binding residues are depicted as spheres. Ligand RMSD is 3.05 Å (b) 12AS (cyan and green) and 1EFV (purple and red), which bind the same ligand: AMP. Ligand RMDS is 0.97 Å.</p></div></div></div></div><div id="__sec14" class="sec sec-last"><h3 id="__sec14title">Computational Time</h3><p id="__p28" class="p p-first-last">It is explained in the report about <em>Apoc</em> that <em>APoc</em> requires 73 seconds as the total computation time (not including input file preparation) for pocket comparison using 1000/1000 randomly selected pairs of pockets from the Subject/Control sets. Thus, APoc consumes 0.037 s for one pair comparison. In contrast, using the same size of pairs randomly selected from the datasets, our new method consumes only 0.00000106 s (=1.06×10<sup>−6</sup>) for one pair comparison with 137-dimensional vectors, although 0.007 s are needed to produce 137-dimensional vectors by counting up the number of triangles from pocket data (not including input file preparation). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f7-13_139/" target="figure" class="fig-table-link figpopup" rid-figpopup="f7-13_139" rid-ob="ob-f7-13_139" co-legend-rid="lgnd_f7-13_139"><span>Figure 7</span></a> shows that the estimation of computational times. If 300,000 known plus 5.2 million unknown pockets are used as pocket data, our new method consumes 200 days just using single thread whereas <em>APoc</em> needs almost 30,000 years which is also just using single thread. We speculate that we can achieve 5.5 million pockets comparison using this new method in a few days when we employ a hundred multi-threads.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="f7-13_139" co-legend-rid="lgnd_f7-13_139"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f7-13_139/" target="figure" rid-figpopup="f7-13_139" rid-ob="ob-f7-13_139"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140309114491936" class="figure"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f7-13_139/" target="figure" rid-figpopup="f7-13_139" rid-ob="ob-f7-13_139"></a><a class="inline_block ts_canvas" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=5042158_13_139_f7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.
Object name is 13_139_f7.jpg" title="Click on image to zoom" class="tileshop" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/13_139_f7.jpg"></a></div><div id="largeobj_idm140309114491936" class="largeobj-link align_right" style=""><a target="object" rel="noopener" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f7-13_139/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f7-13_139"><div><a class="figpopup" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/figure/f7-13_139/" target="figure" rid-figpopup="f7-13_139" rid-ob="ob-f7-13_139">Figure 7</a></div><!--caption a7--><div class="caption"><p id="__p47">Estimated computation time of <em>APoc</em> and our new method. Both the X axis and the Y axis are shown in logarithmic scale.</p></div></div></div></div></div><div id="__sec15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec15title">Conclusion</h2><p id="__p29" class="p p-first">Based on our previous method, for improving the ability to detect similar ligand-binding pockets, we expanded and revised similarity measures of pockets. We observed the effectiveness of those modifications with two different datasets, <em>Ito138</em> and <em>APocS3</em>, in comparison with our previous method. We also found that the effectiveness of the modifications depend on the difficulty of the dataset. These results should be considered for future development of pocket comparison methods. The method proposed herein showed higher detection performance of similar binding pockets than an existing fast sequence order-independent structural alignment method: <em>APoc</em>. Because of its succinct representation, our new method is expected to be useful for large-scale comparison of binding pockets to infer ligands and functions of proteins.</p><div class="boxed-text-box whole_rhythm hide-overflow"><div class="caption"><h3 id="idm140309148161376title">Significance</h3></div><p id="__p30">As the recent increase of protein structure information, comprehensive analysis of protein ligand-binding pockets demands an efficient method for comparing them. Based on our previous method using a vector representation of pockets, we conducted this study to expand and revise similarity measures of pockets by both changing the definition and increasing the number of pocket descriptors, and by using the PAM50 matrix. We investigated the effects of those modifications with two datasets, and found that the new method exhibits slightly higher detection performance of similar pockets than the previous method and an existing accurate alignment-dependent method.</p></div></div><div id="__sec16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec16title">Supplementary Information</h2><!--/article/body/sec/--><div class="sec suppmat" id="s1-13_139"><div class="sup-box half_rhythm" id="d35e1579"><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/bin/13_139_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(752K, pdf)</sup></div></div></div><div id="idm140309107247520" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140309107247520title">Acknowledgment</h2><div class="sec"><p id="__p31">We thank Dr. Jun-Ichi Ito for helpful discussion. We also thank Prof. Tsuda for helpful comments. This study was partially supported by the Platform Project for Supporting in Drug Discovery and Life Science Research (Platform for Drug Discovery, Informatics, and Structural Life Science) from Japan Agency for Medical Research and Development (AMED).</p></div></div><div id="GL" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="GLtitle">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">MDS</td><td>multidimensional scaling</td></tr><tr><td id="G2">TPR</td><td>true positive rate</td></tr><tr><td id="G3">FPR</td><td>false positive rate</td></tr><tr><td id="G4">ROC</td><td>receiver operating characteristic</td></tr><tr><td id="G5">AUC</td><td>area under a ROC curve</td></tr></tbody></table></div></div></div><div id="idm140309107246272" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140309107246272title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn1-13_139"></p><p id="__p32" class="p p-first"><strong>Conflict of Interest</strong></p><p id="__p33" class="p p-last">The authors declare that they have no conflict of interest, financial or otherwise, in relation to this study.</p><p></p><p class="fn sec" id="fn2-13_139"></p><p id="__p34" class="p p-first"><strong>Author Contributions</strong></p><p id="__p35" class="p p-last">T. N. conducted computational experiments and analysis of data, and drafted the manuscript. K. T. supervised the project, contributed to interpretation of data, and wrote the manuscript.</p><p></p></div></div><div id="idm140309108349008" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140309108349008title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b1-13_139">1. <span class="element-citation">Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, et al.  The Protein Data Bank. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>:235–242.</span> <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC102472/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/10592235" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+Protein+Data+Bank&amp;author=HM+Berman&amp;author=J+Westbrook&amp;author=Z+Feng&amp;author=G+Gilliland&amp;author=TN+Bhat&amp;volume=28&amp;publication_year=2000&amp;pages=235-242&amp;pmid=10592235&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b2-13_139">2. <span class="element-citation">Ito J, Ikeda K, Yamada K, Mizuguchi K, Tomii K. PoSSuM v.2.0: data update and a new function for investigating ligand analogs and target proteins of small-molecule drugs. <span><span class="ref-journal">Nucleic Acids Res. </span>2015;<span class="ref-vol">43</span>:D392–D398.</span> <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4383952/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25404129" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=PoSSuM+v.2.0:+data+update+and+a+new+function+for+investigating+ligand+analogs+and+target+proteins+of+small-molecule+drugs&amp;author=J+Ito&amp;author=K+Ikeda&amp;author=K+Yamada&amp;author=K+Mizuguchi&amp;author=K+Tomii&amp;volume=43&amp;publication_year=2015&amp;pages=D392-D398&amp;pmid=25404129&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b3-13_139">3. <span class="element-citation">Konc J, Janežič D. Binding site comparison for function prediction and pharmaceutical discovery. <span><span class="ref-journal">Curr Opin Struct Biol. </span>2014;<span class="ref-vol">25</span>:34–39.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24878342" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Struct+Biol&amp;title=Binding+site+comparison+for+function+prediction+and+pharmaceutical+discovery&amp;author=J+Konc&amp;author=D+Jane%C5%BEi%C4%8D&amp;volume=25&amp;publication_year=2014&amp;pages=34-39&amp;pmid=24878342&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b4-13_139">4. <span class="element-citation">Gao M, Skolnick J. A comprehensive survey of small-molecule binding pockets in proteins. <span><span class="ref-journal">PLoS Comput Biol. </span>2013;<span class="ref-vol">9</span>:e1003302.</span> <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3812058/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/24204237" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput+Biol&amp;title=A+comprehensive+survey+of+small-molecule+binding+pockets+in+proteins&amp;author=M+Gao&amp;author=J+Skolnick&amp;volume=9&amp;publication_year=2013&amp;pages=e1003302&amp;pmid=24204237&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b5-13_139">5. <span class="element-citation">Gao M, Skolnick J. APoc: large-scale identification of similar protein pockets. <span><span class="ref-journal">Bioinformatics. </span>2013;<span class="ref-vol">29</span>:597–604.</span> <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3582269/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/23335017" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=APoc:+large-scale+identification+of+similar+protein+pockets&amp;author=M+Gao&amp;author=J+Skolnick&amp;volume=29&amp;publication_year=2013&amp;pages=597-604&amp;pmid=23335017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b6-13_139">6. <span class="element-citation">Ito J, Tabei Y, Shimizu K, Tomii K, Tsuda K. PDB-scale analysis of known and putative ligand-binding sites with structural sketches. <span><span class="ref-journal">Proteins. </span>2012;<span class="ref-vol">80</span>:747–763.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/22113700" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=PDB-scale+analysis+of+known+and+putative+ligand-binding+sites+with+structural+sketches&amp;author=J+Ito&amp;author=Y+Tabei&amp;author=K+Shimizu&amp;author=K+Tomii&amp;author=K+Tsuda&amp;volume=80&amp;publication_year=2012&amp;pages=747-763&amp;pmid=22113700&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b7-13_139">7. <span class="element-citation">Ito J, Tabei Y, Shimizu K, Tsuda K, Tomii K. PoSSuM: a database of similar protein-ligand binding and putative pockets. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>:D541–D548.</span> <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245044/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/22135290" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=PoSSuM:+a+database+of+similar+protein-ligand+binding+and+putative+pockets&amp;author=J+Ito&amp;author=Y+Tabei&amp;author=K+Shimizu&amp;author=K+Tsuda&amp;author=K+Tomii&amp;volume=40&amp;publication_year=2012&amp;pages=D541-D548&amp;pmid=22135290&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b8-13_139">8. <span class="element-citation">Nakamura T, Tomii K. Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites. <span><span class="ref-journal">Methods. </span>2016;<span class="ref-vol">93</span>:35–40.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26276315" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Protein+ligand-binding+site+comparison+by+a+reduced+vector+representation+derived+from+multidimensional+scaling+of+generalized+description+of+binding+sites&amp;author=T+Nakamura&amp;author=K+Tomii&amp;volume=93&amp;publication_year=2016&amp;pages=35-40&amp;pmid=26276315&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b9-13_139">9. <span class="element-citation">Miyata T, Miyazawa S, Yasunaga T. Two types of amino acid substitutions in protein evolution. <span><span class="ref-journal">J Mol Evol. </span>1979;<span class="ref-vol">12</span>:219–236.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/439147" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Evol&amp;title=Two+types+of+amino+acid+substitutions+in+protein+evolution&amp;author=T+Miyata&amp;author=S+Miyazawa&amp;author=T+Yasunaga&amp;volume=12&amp;publication_year=1979&amp;pages=219-236&amp;pmid=439147&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b10-13_139">10. <span class="element-citation">Dayhoff MO, Schwartz RM. Chapter 22: A model of evolutionary change in proteins. <span><span class="ref-journal">Atlas Protein Seq Struct. </span>1978</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atlas+Protein+Seq+Struct&amp;title=Chapter+22:+A+model+of+evolutionary+change+in+proteins&amp;author=MO+Dayhoff&amp;author=RM+Schwartz&amp;publication_year=1978&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b11-13_139">11. <span class="element-citation">Shawe-Taylor J, Nello C.  <span class="ref-journal">Kernel methods for pattern analysis.</span> Cambridge university press; Cambridge: 2004.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Kernel+methods+for+pattern+analysis&amp;author=J+Shawe-Taylor&amp;author=C+Nello&amp;publication_year=2004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b12-13_139">12. <span class="element-citation">Halko N, Martinsson PG, Tropp JA. Finding structure with randomness: probabilistic algorithms for constructing approximate matrix decompositions. <span><span class="ref-journal">SIAM Rev. </span>2011;<span class="ref-vol">53</span>:217–288.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=SIAM+Rev&amp;title=Finding+structure+with+randomness:+probabilistic+algorithms+for+constructing+approximate+matrix+decompositions&amp;author=N+Halko&amp;author=PG+Martinsson&amp;author=JA+Tropp&amp;volume=53&amp;publication_year=2011&amp;pages=217-288&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b13-13_139">13. <span class="element-citation">Huang B, Schroeder M. LIGSITEcsc: predicting ligand binding sites using the Connolly surface and degree of conservation. <span><span class="ref-journal">BMC Struct Biol. </span>2006;<span class="ref-vol">6</span>:19.</span> <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1601958/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/16995956" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Struct+Biol&amp;title=LIGSITEcsc:+predicting+ligand+binding+sites+using+the+Connolly+surface+and+degree+of+conservation&amp;author=B+Huang&amp;author=M+Schroeder&amp;volume=6&amp;publication_year=2006&amp;pages=19&amp;pmid=16995956&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b14-13_139">14. <span class="element-citation">Weill N, Rognan D. Alignment-free ultra-high-throughput comparison of druggable protein-ligand binding sites. <span><span class="ref-journal">J Chem Inf Model. </span>2010;<span class="ref-vol">50</span>:123–135.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/20058856" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Chem+Inf+Model&amp;title=Alignment-free+ultra-high-throughput+comparison+of+druggable+protein-ligand+binding+sites&amp;author=N+Weill&amp;author=D+Rognan&amp;volume=50&amp;publication_year=2010&amp;pages=123-135&amp;pmid=20058856&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b15-13_139">15. <span class="element-citation">Ghosh A, Uthaiah R, Howard J, Herrmann C, Wolf E. Crystal structure of IIGP1: a paradigm for interferon-inducible p47 resistance GTPases. <span><span class="ref-journal">Mol Cell. </span>2004;<span class="ref-vol">15</span>:727–739.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/15350217" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell&amp;title=Crystal+structure+of+IIGP1:+a+paradigm+for+interferon-inducible+p47+resistance+GTPases&amp;author=A+Ghosh&amp;author=R+Uthaiah&amp;author=J+Howard&amp;author=C+Herrmann&amp;author=E+Wolf&amp;volume=15&amp;publication_year=2004&amp;pages=727-739&amp;pmid=15350217&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b16-13_139">16. <span class="element-citation">Sun HY, Lin SW, Ko TP, Pan JF, Liu CL, Lin CN, et al.  Structure and Mechanism of Helicobacter pylori Fucosyltransferase. A BASIS FOR LIPOPOLYSACCHARIDE VARIATION AND INHIBITOR DESIGN. <span><span class="ref-journal">J Biol Chem. </span>2007;<span class="ref-vol">282</span>:9973–9982.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/17251184" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Structure+and+Mechanism+of+Helicobacter+pylori+Fucosyltransferase.+A+BASIS+FOR+LIPOPOLYSACCHARIDE+VARIATION+AND+INHIBITOR+DESIGN&amp;author=HY+Sun&amp;author=SW+Lin&amp;author=TP+Ko&amp;author=JF+Pan&amp;author=CL+Liu&amp;volume=282&amp;publication_year=2007&amp;pages=9973-9982&amp;pmid=17251184&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b17-13_139">17. <span class="element-citation">Liu ZJ, Sun YJ, Rose J, Chung YJ, Hsiao CD, Chang WR, et al.  The first structure of an aldehyde dehydrogenase reveals novel interactions between NAD and the Rossmann fold. <span><span class="ref-journal">Nat Struct Biol. </span>1997;<span class="ref-vol">4</span>:317–326.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/9095201" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=The+first+structure+of+an+aldehyde+dehydrogenase+reveals+novel+interactions+between+NAD+and+the+Rossmann+fold&amp;author=ZJ+Liu&amp;author=YJ+Sun&amp;author=J+Rose&amp;author=YJ+Chung&amp;author=CD+Hsiao&amp;volume=4&amp;publication_year=1997&amp;pages=317-326&amp;pmid=9095201&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b18-13_139">18. <span class="element-citation">Nakatsu T, Kato H, Oda J. Crystal structure of asparagine synthetase reveals a close evolutionary relationship to class II aminoacyl-tRNA synthetase. <span><span class="ref-journal">Nat Struct Biol. </span>1998;<span class="ref-vol">5</span>:15–19.</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/9437423" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Struct+Biol&amp;title=Crystal+structure+of+asparagine+synthetase+reveals+a+close+evolutionary+relationship+to+class+II+aminoacyl-tRNA+synthetase&amp;author=T+Nakatsu&amp;author=H+Kato&amp;author=J+Oda&amp;volume=5&amp;publication_year=1998&amp;pages=15-19&amp;pmid=9437423&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b19-13_139">19. <span class="element-citation">Roberts DL, Frerman FE, Kim JJP. Three-dimensional structure of human electron transfer flavoprotein to 2.1-Å resolution. <span><span class="ref-journal">Proceedings of the National Academy of Sciences. </span>1996;<span class="ref-vol">93</span>:14355–14360.</span> <span class="nowrap">[<a class="int-reflink" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC26136/">PMC free article</a>]</span> [<a href="https://www.ncbi.nlm.nih.gov/pubmed/8962055" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences&amp;title=Three-dimensional+structure+of+human+electron+transfer+flavoprotein+to+2.1-%C3%85+resolution&amp;author=DL+Roberts&amp;author=FE+Frerman&amp;author=JJP+Kim&amp;volume=93&amp;publication_year=1996&amp;pages=14355-14360&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#idm140309110378112title">Abstract</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#__sec1title">Materials and Methods</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#__sec8title">Results and Discussion</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#__sec15title">Conclusion</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#__sec16title">Supplementary Information</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#idm140309107247520title">Acknowledgment</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#GLtitle">Abbreviations</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#idm140309107246272title">Footnotes</a></li><li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#idm140309108349008title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Biophysics and Physicobiology</span> are provided here courtesy of <strong>The Biophysical Society of Japan</strong></div><hr></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/?report=reader">PubReader</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/pdf/13_139.pdf">PDF (1.2M)</a></li> | <li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" data-citationid="PMC5042158" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Cite</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5042158%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5042158%2F&amp;text=Effects%20of%20the%20difference%20in%20similarity%20measures%20on%20the%20comparison%20of%20ligand-binding%20pockets%20using%20a%20reduced%20vector%20representation%20of%20pockets" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5042158%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27924268" data-aiid="5042158" data-aid="5042158" data-iid="276315" data-domainid="2913" data-domain="bpap" data-accid="PMC5042158" data-md5="3db0e805c4a1574d4c61aacdd4301433">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-4" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-4" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input type="hidden" name="absid" id="absid" value="5042158"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:&#39;#favUL&#39;,starExists:true,createCollectionUrl:&#39;#&#39;,manageCollectionsUrl:&#39;/myncbi/collections/&#39;"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="https://www.ncbi.nlm.nih.gov/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input type="submit" class="button_apply" name="addtocollection" value="1"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-5" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-5" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/26276315" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites.</a><span class="source">[Methods. 2016]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-6" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Nakamura T, Tomii K. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Methods. 2016 Jan 15; 93:35-40. Epub 2015 Aug 11.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/20175916" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A new protein binding pocket similarity measure based on comparison of clouds of atoms in 3D: application to ligand prediction.</a><span class="source">[BMC Bioinformatics. 2010]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Hoffmann B, Zaslavskiy M, Vert JP, Stoven V. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">BMC Bioinformatics. 2010 Feb 22; 11:99. Epub 2010 Feb 22.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/23335017" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">APoc: large-scale identification of similar protein pockets.</a><span class="source">[Bioinformatics. 2013]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Gao M, Skolnick J. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Bioinformatics. 2013 Mar 1; 29(5):597-604. Epub 2013 Jan 17.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/25690787" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Implications of the small number of distinct ligand binding pockets in proteins for drug discovery, evolution and biochemical function.</a><span class="source">[Bioorg Med Chem Lett. 2015]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Skolnick J, Gao M, Roy A, Srinivasan B, Zhou H. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Bioorg Med Chem Lett. 2015 Mar 15; 25(6):1163-70. Epub 2015 Feb 3.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/21963969" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">[Development and validation of programs for ligand-binding-pocket search].</a><span class="source">[Yakugaku Zasshi. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Oda A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Yakugaku Zasshi. 2011; 131(10):1429-35. </em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=27924268&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=27924268&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-11" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-11" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6224041/" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A benchmark driven guide to binding site comparison: An exhaustive evaluation using tailor-made data sets (ProSPECCTs)</a><span class="source">[PLoS Computational Biology. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-12" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Ehrt C, Brinkjost T, Koch O. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS Computational Biology. 2018 Nov 8; 14(11): e1006483</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5845264/" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Comparative assessment of strategies to identify similar ligand-binding pockets in proteins</a><span class="source">[BMC Bioinformatics. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Govindaraj RG, Brylinski M. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">BMC Bioinformatics. 2018 Mar 9; 19: 91</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042156/" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Preface of Special Issue “Protein-Ligand Interactions”</a><span class="source">[Biophysics and Physicobiology....]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Yura K. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Biophysics and Physicobiology. 2016 Jul 14; 13: 85-86</em>
</div>
</li>
</ul>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/citedby/" ref="reftype=relart&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-15" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-15" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=5042158" ref="reftype=MedGen&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pubmed/27924268/" ref="reftype=PubMed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="https://www.ncbi.nlm.nih.gov/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=5042158" ref="reftype=Taxonomy&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-18" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-19" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-19" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState(&#39;ClearHT&#39;)">Clear</a><a href="javascript:historyDisplayState(&#39;HTOff&#39;)" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState(&#39;HTOn&#39;)" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=613904b8ff84a16748900ab1">Effects of the difference in similarity measures on the comparison of ligand-bin...</a><div class="ralinkpop offscreen_noflow">Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets<div class="brieflinkpopdesc">Biophysics and Physicobiology. 2016; 13()139</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=613195f60202827c7993d0a2">Computational models identify several FDA approved or experimental drugs as puta...</a><div class="ralinkpop offscreen_noflow">Computational models identify several FDA approved or experimental drugs as putative agents against SARS-CoV-2<div class="brieflinkpopdesc">ChemRxiv. ; ()</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="https://www.ncbi.nlm.nih.gov/portal/utils/pageresolver.fcgi?recordid=6131842e0202827c7900359c">Finding low-energy conformations of lattice protein models by quantum annealing</a><div class="ralinkpop offscreen_noflow">Finding low-energy conformations of lattice protein models by quantum annealing<div class="brieflinkpopdesc">Scientific Reports. 2012; 2()</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState(&#39;HTOn&#39;)">Turn recording back on</a></p>
<a class="seemore" href="https://www.ncbi.nlm.nih.gov/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p2" name="crb--__p2" rid="__p2" style="position: absolute; visibility: visible; width: 369.734px; top: 1162px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-20"><ul>
<li class="two_line" reference_id="b1-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/10592235/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Protein Data Bank.</a><span class="one_line_source">[Nucleic Acids Res.  2000]</span><div class="alt-note">
<div class="authors">Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE</div>
<div class="citation">Nucleic Acids Res. 2000 Jan 1; 28(1):235-42.</div>
</div>
</li>
<li class="two_line" reference_id="b2-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25404129/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PoSSuM v.2.0: data update and a new function for investigating ligand analogs and target proteins of small-molecule drugs.</a><span class="one_line_source">[Nucleic Acids Res.  2015]</span><div class="alt-note">
<div class="authors">Ito J, Ikeda K, Yamada K, Mizuguchi K, Tomii K</div>
<div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D392-8.</div>
</div>
</li>
<li class="two_line" reference_id="b3-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/24878342/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Binding site comparison for function prediction and pharmaceutical discovery.</a><span class="one_line_source">[Curr Opin Struct Biol.  2014]</span><div class="alt-note">
<div class="authors">Konc J, Janežič D</div>
<div class="citation">Curr Opin Struct Biol. 2014 Apr; 25():34-9.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p3" name="crb--__p3" rid="__p3" style="position: absolute; visibility: visible; width: 369.734px; top: 1371px; display: none;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-21"><ul>
<li class="two_line" reference_id="b4-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/24204237/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A comprehensive survey of small-molecule binding pockets in proteins.</a><span class="one_line_source">[PLoS Comput Biol.  2013]</span><div class="alt-note">
<div class="authors">Gao M, Skolnick J</div>
<div class="citation">PLoS Comput Biol. 2013 Oct; 9(10):e1003302.</div>
</div>
</li>
<li class="two_line" reference_id="b5-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23335017/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">APoc: large-scale identification of similar protein pockets.</a><span class="one_line_source">[Bioinformatics.  2013]</span><div class="alt-note">
<div class="authors">Gao M, Skolnick J</div>
<div class="citation">Bioinformatics. 2013 Mar 1; 29(5):597-604.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p4" name="crb--__p4" rid="__p4" style="position: absolute; visibility: visible; width: 369.734px; top: 1668px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-22"><ul>
<li class="two_line" reference_id="b6-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22113700/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PDB-scale analysis of known and putative ligand-binding sites with structural sketches.</a><span class="one_line_source">[Proteins.  2012]</span><div class="alt-note">
<div class="authors">Ito J, Tabei Y, Shimizu K, Tomii K, Tsuda K</div>
<div class="citation">Proteins. 2012 Mar; 80(3):747-63.</div>
</div>
</li>
<li class="two_line" reference_id="b2-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/25404129/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PoSSuM v.2.0: data update and a new function for investigating ligand analogs and target proteins of small-molecule drugs.</a><span class="one_line_source">[Nucleic Acids Res.  2015]</span><div class="alt-note">
<div class="authors">Ito J, Ikeda K, Yamada K, Mizuguchi K, Tomii K</div>
<div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D392-8.</div>
</div>
</li>
<li class="two_line" reference_id="b7-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22135290/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PoSSuM: a database of similar protein-ligand binding and putative pockets.</a><span class="one_line_source">[Nucleic Acids Res.  2012]</span><div class="alt-note">
<div class="authors">Ito J, Tabei Y, Shimizu K, Tsuda K, Tomii K</div>
<div class="citation">Nucleic Acids Res. 2012 Jan; 40(Database issue):D541-8.</div>
</div>
</li>
<li class="two_line" reference_id="b8-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26276315/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites.</a><span class="one_line_source">[Methods.  2016]</span><div class="alt-note">
<div class="authors">Nakamura T, Tomii K</div>
<div class="citation">Methods. 2016 Jan 15; 93():35-40.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p6" name="crb--__p6" rid="__p6" style="position: absolute; visibility: visible; width: 369.734px; top: 2254px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-23"><ul>
<li class="two_line" reference_id="b8-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26276315/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites.</a><span class="one_line_source">[Methods.  2016]</span><div class="alt-note">
<div class="authors">Nakamura T, Tomii K</div>
<div class="citation">Methods. 2016 Jan 15; 93():35-40.</div>
</div>
</li>
<li class="two_line" reference_id="b9-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/439147/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Two types of amino acid substitutions in protein evolution.</a><span class="one_line_source">[J Mol Evol.  1979]</span><div class="alt-note">
<div class="authors">Miyata T, Miyazawa S, Yasunaga T</div>
<div class="citation">J Mol Evol. 1979 Mar 15; 12(3):219-36.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p7" name="crb--__p7" rid="__p7" style="position: absolute; visibility: visible; width: 369.734px; top: 2722px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul><li class="two_line" reference_id="b8-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26276315/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites.</a><span class="one_line_source">[Methods.  2016]</span><div class="alt-note">
<div class="authors">Nakamura T, Tomii K</div>
<div class="citation">Methods. 2016 Jan 15; 93():35-40.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p16" name="crb--__p16" rid="__p16" style="position: absolute; visibility: visible; width: 369.734px; top: 4542px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul>
<li class="two_line" reference_id="b8-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26276315/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites.</a><span class="one_line_source">[Methods.  2016]</span><div class="alt-note">
<div class="authors">Nakamura T, Tomii K</div>
<div class="citation">Methods. 2016 Jan 15; 93():35-40.</div>
</div>
</li>
<li class="two_line" reference_id="b5-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/23335017/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">APoc: large-scale identification of similar protein pockets.</a><span class="one_line_source">[Bioinformatics.  2013]</span><div class="alt-note">
<div class="authors">Gao M, Skolnick J</div>
<div class="citation">Bioinformatics. 2013 Mar 1; 29(5):597-604.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p17" name="crb--__p17" rid="__p17" style="position: absolute; visibility: visible; width: 369.734px; top: 5314px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul><li class="two_line" reference_id="b13-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/16995956/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">LIGSITEcsc: predicting ligand binding sites using the Connolly surface and degree of conservation.</a><span class="one_line_source">[BMC Struct Biol.  2006]</span><div class="alt-note">
<div class="authors">Huang B, Schroeder M</div>
<div class="citation">BMC Struct Biol. 2006 Sep 24; 6():19.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p21" name="crb--__p21" rid="__p21" style="position: absolute; visibility: visible; width: 369.734px; top: 6441px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul><li class="two_line" reference_id="b8-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26276315/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites.</a><span class="one_line_source">[Methods.  2016]</span><div class="alt-note">
<div class="authors">Nakamura T, Tomii K</div>
<div class="citation">Methods. 2016 Jan 15; 93():35-40.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p24" name="crb--__p24" rid="__p24" style="position: absolute; visibility: visible; width: 369.734px; top: 7345px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul>
<li class="two_line" reference_id="b6-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/22113700/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PDB-scale analysis of known and putative ligand-binding sites with structural sketches.</a><span class="one_line_source">[Proteins.  2012]</span><div class="alt-note">
<div class="authors">Ito J, Tabei Y, Shimizu K, Tomii K, Tsuda K</div>
<div class="citation">Proteins. 2012 Mar; 80(3):747-63.</div>
</div>
</li>
<li class="two_line" reference_id="b14-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/20058856/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Alignment-free ultra-high-throughput comparison of druggable protein-ligand binding sites.</a><span class="one_line_source">[J Chem Inf Model.  2010]</span><div class="alt-note">
<div class="authors">Weill N, Rognan D</div>
<div class="citation">J Chem Inf Model. 2010 Jan; 50(1):123-35.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p26" name="crb--__p26" rid="__p26" style="position: absolute; visibility: visible; width: 369.734px; top: 8421px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul>
<li class="two_line" reference_id="b15-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/15350217/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Crystal structure of IIGP1: a paradigm for interferon-inducible p47 resistance GTPases.</a><span class="one_line_source">[Mol Cell.  2004]</span><div class="alt-note">
<div class="authors">Ghosh A, Uthaiah R, Howard J, Herrmann C, Wolf E</div>
<div class="citation">Mol Cell. 2004 Sep 10; 15(5):727-39.</div>
</div>
</li>
<li class="two_line" reference_id="b16-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/17251184/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Structure and mechanism of Helicobacter pylori fucosyltransferase. A basis for lipopolysaccharide variation and inhibitor design.</a><span class="one_line_source">[J Biol Chem.  2007]</span><div class="alt-note">
<div class="authors">Sun HY, Lin SW, Ko TP, Pan JF, Liu CL, Lin CN, Wang AH, Lin CH</div>
<div class="citation">J Biol Chem. 2007 Mar 30; 282(13):9973-9982.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p27" name="crb--__p27" rid="__p27" style="position: absolute; visibility: visible; width: 369.734px; top: 8674px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul>
<li class="two_line" reference_id="b17-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9095201/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The first structure of an aldehyde dehydrogenase reveals novel interactions between NAD and the Rossmann fold.</a><span class="one_line_source">[Nat Struct Biol.  1997]</span><div class="alt-note">
<div class="authors">Liu ZJ, Sun YJ, Rose J, Chung YJ, Hsiao CD, Chang WR, Kuo I, Perozich J, Lindahl R, Hempel J, Wang BC</div>
<div class="citation">Nat Struct Biol. 1997 Apr; 4(4):317-26.</div>
</div>
</li>
<li class="two_line" reference_id="b18-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/9437423/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Crystal structure of asparagine synthetase reveals a close evolutionary relationship to class II aminoacyl-tRNA synthetase.</a><span class="one_line_source">[Nat Struct Biol.  1998]</span><div class="alt-note">
<div class="authors">Nakatsu T, Kato H, Oda J</div>
<div class="citation">Nat Struct Biol. 1998 Jan; 5(1):15-9.</div>
</div>
</li>
<li class="two_line" reference_id="b19-13_139">
<a href="https://www.ncbi.nlm.nih.gov/pubmed/8962055/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Three-dimensional structure of human electron transfer flavoprotein to 2.1-A resolution.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  1996]</span><div class="alt-note">
<div class="authors">Roberts DL, Frerman FE, Kim JJ</div>
<div class="citation">Proc Natl Acad Sci U S A. 1996 Dec 10; 93(25):14355-60.</div>
</div>
</li>
</ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-b1-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The Protein Data Bank.<div class="authors">Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE</div><div class="citation">Nucleic Acids Res. 2000 Jan 1; 28(1):235-42.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/10592235/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b1-13_139">Ref list</a>]</p></div><div id="body-link-popper-b2-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">PoSSuM v.2.0: data update and a new function for investigating ligand analogs and target proteins of small-molecule drugs.<div class="authors">Ito J, Ikeda K, Yamada K, Mizuguchi K, Tomii K</div><div class="citation">Nucleic Acids Res. 2015 Jan; 43(Database issue):D392-8.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/25404129/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b2-13_139">Ref list</a>]</p></div><div id="body-link-popper-b3-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true"><span class="invert">Review</span> Binding site comparison for function prediction and pharmaceutical discovery.<div class="authors">Konc J, Janežič D</div><div class="citation">Curr Opin Struct Biol. 2014 Apr; 25():34-9.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/24878342/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b3-13_139">Ref list</a>]</p></div><div id="body-link-popper-b4-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">A comprehensive survey of small-molecule binding pockets in proteins.<div class="authors">Gao M, Skolnick J</div><div class="citation">PLoS Comput Biol. 2013 Oct; 9(10):e1003302.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/24204237/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b4-13_139">Ref list</a>]</p></div><div id="body-link-popper-b5-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">APoc: large-scale identification of similar protein pockets.<div class="authors">Gao M, Skolnick J</div><div class="citation">Bioinformatics. 2013 Mar 1; 29(5):597-604.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/23335017/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b5-13_139">Ref list</a>]</p></div><div id="body-link-popper-b6-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">PDB-scale analysis of known and putative ligand-binding sites with structural sketches.<div class="authors">Ito J, Tabei Y, Shimizu K, Tomii K, Tsuda K</div><div class="citation">Proteins. 2012 Mar; 80(3):747-63.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/22113700/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b6-13_139">Ref list</a>]</p></div><div id="body-link-popper-b7-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">PoSSuM: a database of similar protein-ligand binding and putative pockets.<div class="authors">Ito J, Tabei Y, Shimizu K, Tsuda K, Tomii K</div><div class="citation">Nucleic Acids Res. 2012 Jan; 40(Database issue):D541-8.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/22135290/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b7-13_139">Ref list</a>]</p></div><div id="body-link-popper-b8-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Protein ligand-binding site comparison by a reduced vector representation derived from multidimensional scaling of generalized description of binding sites.<div class="authors">Nakamura T, Tomii K</div><div class="citation">Methods. 2016 Jan 15; 93():35-40.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/26276315/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b8-13_139">Ref list</a>]</p></div><div id="body-link-popper-b9-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Two types of amino acid substitutions in protein evolution.<div class="authors">Miyata T, Miyazawa S, Yasunaga T</div><div class="citation">J Mol Evol. 1979 Mar 15; 12(3):219-36.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/439147/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b9-13_139">Ref list</a>]</p></div><div id="body-link-popper-b10-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">10. <span class="element-citation">Dayhoff MO, Schwartz RM. Chapter 22: A model of evolutionary change in proteins. <span><span class="ref-journal">Atlas Protein Seq Struct. </span>1978</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atlas+Protein+Seq+Struct&amp;title=Chapter+22:+A+model+of+evolutionary+change+in+proteins&amp;author=MO+Dayhoff&amp;author=RM+Schwartz&amp;publication_year=1978&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b10-13_139">Ref list</a>]</div><div id="body-link-popper-b11-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">11. <span class="element-citation">Shawe-Taylor J, Nello C.  <span class="ref-journal">Kernel methods for pattern analysis.</span> Cambridge university press; Cambridge: 2004.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Kernel+methods+for+pattern+analysis&amp;author=J+Shawe-Taylor&amp;author=C+Nello&amp;publication_year=2004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b11-13_139">Ref list</a>]</div><div id="body-link-popper-b12-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">12. <span class="element-citation">Halko N, Martinsson PG, Tropp JA. Finding structure with randomness: probabilistic algorithms for constructing approximate matrix decompositions. <span><span class="ref-journal">SIAM Rev. </span>2011;<span class="ref-vol">53</span>:217–288.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=SIAM+Rev&amp;title=Finding+structure+with+randomness:+probabilistic+algorithms+for+constructing+approximate+matrix+decompositions&amp;author=N+Halko&amp;author=PG+Martinsson&amp;author=JA+Tropp&amp;volume=53&amp;publication_year=2011&amp;pages=217-288&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b12-13_139">Ref list</a>]</div><div id="body-link-popper-b13-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">LIGSITEcsc: predicting ligand binding sites using the Connolly surface and degree of conservation.<div class="authors">Huang B, Schroeder M</div><div class="citation">BMC Struct Biol. 2006 Sep 24; 6():19.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/16995956/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b13-13_139">Ref list</a>]</p></div><div id="body-link-popper-b14-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Alignment-free ultra-high-throughput comparison of druggable protein-ligand binding sites.<div class="authors">Weill N, Rognan D</div><div class="citation">J Chem Inf Model. 2010 Jan; 50(1):123-35.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/20058856/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b14-13_139">Ref list</a>]</p></div><div id="body-link-popper-b15-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Crystal structure of IIGP1: a paradigm for interferon-inducible p47 resistance GTPases.<div class="authors">Ghosh A, Uthaiah R, Howard J, Herrmann C, Wolf E</div><div class="citation">Mol Cell. 2004 Sep 10; 15(5):727-39.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/15350217/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b15-13_139">Ref list</a>]</p></div><div id="body-link-popper-b16-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Structure and mechanism of Helicobacter pylori fucosyltransferase. A basis for lipopolysaccharide variation and inhibitor design.<div class="authors">Sun HY, Lin SW, Ko TP, Pan JF, Liu CL, Lin CN, Wang AH, Lin CH</div><div class="citation">J Biol Chem. 2007 Mar 30; 282(13):9973-9982.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/17251184/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b16-13_139">Ref list</a>]</p></div><div id="body-link-popper-b17-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">The first structure of an aldehyde dehydrogenase reveals novel interactions between NAD and the Rossmann fold.<div class="authors">Liu ZJ, Sun YJ, Rose J, Chung YJ, Hsiao CD, Chang WR, Kuo I, Perozich J, Lindahl R, Hempel J, Wang BC</div><div class="citation">Nat Struct Biol. 1997 Apr; 4(4):317-26.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9095201/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b17-13_139">Ref list</a>]</p></div><div id="body-link-popper-b18-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Crystal structure of asparagine synthetase reveals a close evolutionary relationship to class II aminoacyl-tRNA synthetase.<div class="authors">Nakatsu T, Kato H, Oda J</div><div class="citation">Nat Struct Biol. 1998 Jan; 5(1):15-9.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/9437423/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b18-13_139">Ref list</a>]</p></div><div id="body-link-popper-b19-13_139" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">Three-dimensional structure of human electron transfer flavoprotein to 2.1-A resolution.<div class="authors">Roberts DL, Frerman FE, Kim JJ</div><div class="citation">Proc Natl Acad Sci U S A. 1996 Dec 10; 93(25):14355-60.</div><p>[<a href="https://www.ncbi.nlm.nih.gov/pubmed/8962055/" ref="reftype=pubmed&amp;article-id=5042158&amp;issue-id=276315&amp;journal-id=2913&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PubMed</a>] [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#b19-13_139">Ref list</a>]</p></div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2021-09-08T14:45:12-04:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.52&amp;Host=ptpmc101&amp;ncbi_phid=8A1A6F1013903AB10000000000600060&amp;ncbi_session=8A1B9F7B118DA681_0091SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC5042158%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov/" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov/" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov/" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov/" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov/">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="https://www.ncbi.nlm.nih.gov/home/about/policies.shtml">Policies and Guidelines</a> | <a href="https://www.ncbi.nlm.nih.gov/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/InstrumentPageStarterJS.js"> </script>    
    <script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 63.5938px; left: 419.188px; width: 669.656px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5042158/#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div><div id="ZN_dikYWqsjiUWN0Q5"></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=bpap&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC5042158/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- 8A1B9F7B118DA681_0091SID /projects/PMC/PMCViewer@4.52 ptpmc101 v4.1.r635708 Mon, Aug 09 2021 01:08:56 -->
<span id="portal-csrf-token" style="display:none" data-token="8A1B9F7B118DA681_0091SID"></span>

<script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/4065628.js" snapshot="pmc"></script>
<script type="text/javascript" src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/saved_resource"></script><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: bold; font-style: italic; font-family: Helvetica;"></div></div><script src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/CoreModule.js" defer=""></script><div class="ui-ncbipopper-arrow-image ui-ncbipopper-arrow-right" style="display: none; top: -100px; left: -100px; opacity: 1;">&nbsp;</div></body><div id="cVim-status-bar" style="top: 0px;"></div><iframe src="./Effects of the difference in similarity measures on the comparison of ligand-binding pockets using a reduced vector representation of pockets_files/cmdline_frame.html" id="cVim-command-frame"></iframe></html>